const data = [
	{
		"datasets": [
			{
				"folder": "",
				"file_count": 720,
				"description": "H4_1",
				"size": 6040166400
			}
		],
		"name": "Structure-Informed Design of an Enzymatically Inactive Vaccine Component for Group A Streptococcus",
		"date_modified": "2014-03-24 05:28:04.911373+00:00",
		"datafile_count": 720,
		"description": {
			"admins": [
				"steve.androulakis"
			],
			"institution": "University of Queensland, Australian Synchrotron",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 1,
		"authors": [
			"Anna Henningham",
			"Daniel J. Ericsson",
			"Karla Langer",
			"Lachlan Casey",
			"Blagojce Jovcevski",
			"G. Singh Chhatwal",
			"J. Andrew Aquilina",
			"Michael R. Batzloff",
			"Bostjan Kobe",
			"Mark Walker"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Streptococcus pyogenes"
					},
					{
						"name": "PDB ID",
						"value": "4BOF"
					},
					{
						"name": "Resolution",
						"value": "2.48\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.244"
					},
					{
						"name": "R-Value",
						"value": "0.213 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">4BOF:D|PDBID|CHAIN|SEQUENCE MTAQTPIHVYSEIGKLKKVLLHRPGKEIENLMPDYLERLLFDDIPFLEDAQKEHDAFAQALRDEGIEVLYLETLAAESLV TPEIREAFIDEYLSEANIRGRATKKAIRELLMAIEDNQELIEKTMAGVQKSELPEIPASEKGLTDLVESNYPFAIDPMPN LYFTRDPFATIGTGVSLNHMFSETRNRETLYGKYIFTHHPIYGGGKVPMVYDRNETTRIEGGDELVLSKDVLAVGISQRT DAASIEKLLVNIFKQNLGFKKVLAFEFANNRKFMHLDTVFTMVDYDKFTIHPEIEGDLRVYSVTYDNEELHIVEEKGDLA ELLAANLGVEKVDLIRCGGDNLVAAGREQWNDGSNTLTIAPGVVVVYNRNTITNAILESKGLKLIKIHGSELVRGRGGPR CMSMPFEREDI"
					},
					{
						"name": "Space Group",
						"value": "P 1"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 87.91, b = 92.74, c = 120.97),(a = 96.15, b = 90.27, c = 100.13)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=4BOF"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX1"
					},
					{
						"name": "EPN",
						"value": "4014c"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://dx.doi.org/10.1128/mBio.00509-13"
					},
					{
						"name": "Notes",
						"value": "mBio"
					},
					{
						"name": "Title",
						"value": "Structure-Informed Design of an Enzymatically Inactive Vaccine Component for Group A Streptococcus"
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			}
		],
		"id": 222,
		"size": 6040166400
	},
	{
		"datasets": [
			{
				"folder": "",
				"file_count": 180,
				"description": "D6_2",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "D6_1",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 10,
				"description": "D6_0",
				"size": 188748800
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "C6_1",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "C5_1",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "B7_2",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "B6_2",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 165,
				"description": "A8_2",
				"size": 3114355200
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "A6_2",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "A5_2",
				"size": 3397478400
			}
		],
		"name": "Crystal Structures of Three Classes of Non-Steroidal Anti-Inflammatory Drugs in Complex with Aldo-Keto Reductase 1C3",
		"date_modified": "2014-03-24 05:27:55.791952+00:00",
		"datafile_count": 1615,
		"description": {
			"admins": [
				"steve.androulakis"
			],
			"institution": "University of Auckland, Australian Synchrotron",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 10,
		"authors": [
			"Yuliana Yosaatmadja",
			"Jack Flanagan"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3432z"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://dx.doi.org/10.1371/journal.pone.0043965"
					},
					{
						"name": "Notes",
						"value": "PLOS One"
					},
					{
						"name": "Title",
						"value": "Crystal Structures of Three Classes of Non-Steroidal Anti-Inflammatory Drugs in Complex with Aldo-Keto Reductase 1C3"
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3R94"
					},
					{
						"name": "Resolution",
						"value": "2.01\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.221"
					},
					{
						"name": "R-Value",
						"value": "0.178 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3R94:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 56.87, b = 63.77, c = 95.67),(a = 90.00, b = 90.00, c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3R94"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3R8G"
					},
					{
						"name": "Resolution",
						"value": "1.8\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.213"
					},
					{
						"name": "R-Value",
						"value": "0.176 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3R8G:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 57.79 , b = 64.53 , c = 96.18 ),(a = 90.00 , b = 90.00 , c = 90.00  )"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3R8G"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3UFY"
					},
					{
						"name": "Resolution",
						"value": "1.9\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.194"
					},
					{
						"name": "R-Value",
						"value": "0.174 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3UFY:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 58.55, b = 64.58, c = 96.56),(a = 90.00, b = 90.00, c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3UFY"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3UG8"
					},
					{
						"name": "Resolution",
						"value": "1.73\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2"
					},
					{
						"name": "R-Value",
						"value": "0.174 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3UG8:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 56.66 , b = 63.94 , c = 96.52 ),(a = 90.00 , b = 90.00 , c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3UG8"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3R8H"
					},
					{
						"name": "Resolution",
						"value": "1.9\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.204"
					},
					{
						"name": "R-Value",
						"value": "0.166 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3R8H:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 57.29 , b = 64.04 , c = 95.98 ),(a = 90.00 , b = 90.00 , c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3R8H"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3R7M"
					},
					{
						"name": "Resolution",
						"value": "2.1\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2323"
					},
					{
						"name": "R-Value",
						"value": "0.185 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3R7M:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 58.38 , b = 64.48 , c = 96.85 ),(a = 90.00 , b = 90.00 , c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3R7M"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3R58"
					},
					{
						"name": "Resolution",
						"value": "2.3\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.22"
					},
					{
						"name": "R-Value",
						"value": "0.182 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3R58:A|PDBID|CHAIN|SEQUENCE MDSKHQCVKLNDGHFMPVLGFGTYAPPEVPRSKALEVTKLAIEAGFRHIDSAHLYNNEEQVGLAIRSKIADGSVKREDIF YTSKLWSTFHRPELVRPALENSLKKAQLDYVDLYLIHSPMSLKPGEELSPTDENGKVIFDIVDLCTTWEAMEKCKDAGLA KSIGVSNFNRRQLEMILNKPGLKYKPVCNQVECHPYFNRSKLLDFCKSKDIVLVAYSALGSQRD"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 58.76 , b = 64.93 , c = 96.27),(a = 90.00 , b = 90.00 , c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore/explore.do?structureId=3r58"
					}
				],
				"name": "Protein Data Bank"
			}
		],
		"id": 251,
		"size": 30482931200
	},
	{
		"datasets": [
			{
				"folder": "",
				"file_count": 720,
				"description": "G6_3",
				"size": 13589913600
			},
			{
				"folder": "",
				"file_count": 720,
				"description": "G6_2",
				"size": 13589913600
			},
			{
				"folder": "",
				"file_count": 720,
				"description": "G6_1",
				"size": 13589913600
			},
			{
				"folder": "",
				"file_count": 720,
				"description": "D6_1",
				"size": 13589913600
			}
		],
		"name": "Structure of the heterodimer of human NONO and paraspeckle protein component 1",
		"date_modified": "2014-03-24 05:27:47.031404+00:00",
		"datafile_count": 2880,
		"description": {
			"admins": [
				"steve.androulakis"
			],
			"institution": "University of Western Australia, Australian Synchrotron",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 4,
		"authors": [
			"Charlie Bond ",
			"Daniel Passon ",
			"Mihwa Lee "
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "1562"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Homo sapiens"
					},
					{
						"name": "PDB ID",
						"value": "3SDE"
					},
					{
						"name": "Resolution",
						"value": "1.9\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.228"
					},
					{
						"name": "R-Value",
						"value": "0.185 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3SDE:A|PDBID|CHAIN|SEQUENCE GMGFTIDIKSFLKPGEKTYTQRCRLFVGNLPTDITEEDFKRLFERYGEPSEVFINRDRGFGFIRLESRTLAEIAKAELDG TILKSRPLRIRFATHGAALTVKNLSPVVSNELLEQAFSQFGPVEKAVVVVDDRGRATGKGFVEFAAKPPARKALERCGDG AFLLTTTPRPVIVEPMEQFDDEDGLPEKLMQKTQQYHKEREQPPRFAQPGTFEFEYASRWKALDEMEKQQREQVDRNIRE AKEKLEAEMEAARHEHQLMLM"
					},
					{
						"name": "Space Group",
						"value": "C 2"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 90.90, b = 67.18, c = 94.08),(a = 90.00 , b = 99.96 , c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore.do?structureId=3SDE"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://www.ncbi.nlm.nih.gov/pubmed/22102035"
					},
					{
						"name": "Notes",
						"value": "Acta crystallographica. Section F, Structural biology and crystallization communications."
					},
					{
						"name": "Title",
						"value": "Crystallization of a paraspeckle protein PSPC1-NONO heterodimer."
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://www.ncbi.nlm.nih.gov/pubmed/22416126"
					},
					{
						"name": "Notes",
						"value": "Proceedings of the National Academy of Sciences of the United States of America."
					},
					{
						"name": "Title",
						"value": "Structure of the heterodimer of human NONO and paraspeckle protein component 1 and analysis of its role in subnuclear body formation."
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			}
		],
		"id": 257,
		"size": 54359654400
	},
	{
		"datasets": [
			{
				"folder": "",
				"file_count": 94,
				"description": "2K3e_5",
				"size": 1774238720
			},
			{
				"folder": "",
				"file_count": 180,
				"description": "2K3d_4",
				"size": 3397478400
			},
			{
				"folder": "",
				"file_count": 120,
				"description": "2K7c_3",
				"size": 2264985600
			},
			{
				"folder": "",
				"file_count": 181,
				"description": "2K3c_3",
				"size": 3416353280
			}
		],
		"name": "Derivatives for structure solution of the peripheral stalk from T.thermophilus A-ATPase",
		"date_modified": "2014-03-24 05:28:13.456665+00:00",
		"datafile_count": 575,
		"description": {
			"admins": [
				"steve.androulakis"
			],
			"institution": "Victor Chang Cardiac Research Institute, Australian Synchrotron",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 4,
		"authors": [
			"Alastair Stewart",
			"Daniela Stock"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Thermus thermophilus HB8"
					},
					{
						"name": "PDB ID",
						"value": "3V6I"
					},
					{
						"name": "Resolution",
						"value": "2.25\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.263"
					},
					{
						"name": "R-Value",
						"value": "0.225 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3V6I:B|PDBID|CHAIN|SEQUENCE GMGGLGLIKSLAEKEKQLLERLEAAKKEAEERVKRAEAEAKALLEEAEAKAKALEAQYRERERAETEALLARYRERAEAE AKAVREKAMARLDEAVALVLKEVLP"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 2"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 102.51, b = 208.69, c = 36.90),(a = 90.00, b = 90.00, c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.rcsb.org/pdb/explore.do?structureId=3V6I"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3511"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://www.ncbi.nlm.nih.gov/pubmed/22353718"
					},
					{
						"name": "Notes",
						"value": "Nature Communications"
					},
					{
						"name": "Title",
						"value": "The dynamic stator stalk of rotary ATPases."
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			}
		],
		"id": 210,
		"size": 10853056000
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 1,
				"description": "testcrystal_0",
				"size": 8389632
			},
			{
				"folder": null,
				"file_count": 180,
				"description": "testcrystal_1",
				"size": 1510133760
			}
		],
		"name": "Test Experiment MX1cal20130930",
		"date_modified": "2014-03-24 05:28:39.618975+00:00",
		"datafile_count": 181,
		"description": {
			"admins": [
				"synchrotron"
			],
			"institution": "Australian Synchrotron",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [],
		"metadata": [
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX1"
					},
					{
						"name": "EPN",
						"value": "MX1cal20130930"
					}
				],
				"name": "Experiment ID"
			}
		],
		"id": 186,
		"size": 1518523392
	},
	{
		"datasets": [
			{
				"folder": "",
				"file_count": 220,
				"description": "smax_mg_D4_3",
				"size": 4152473600
			}
		],
		"name": "Defining the interaction of perforin with calcium and the phospholipid membrane",
		"date_modified": "2014-03-25 00:39:58.705267+00:00",
		"datafile_count": 220,
		"description": {
			"admins": [
				"steve.androulakis"
			],
			"institution": "Monash University, Australian Synchrotron",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 1,
		"authors": [
			"Daouda A.K. Traore",
			"James C Whisstock"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Scophthalmus maximus"
					},
					{
						"name": "PDB ID",
						"value": "3W56"
					},
					{
						"name": "Resolution",
						"value": "1.6\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.251"
					},
					{
						"name": "R-Value",
						"value": "0.214 (obs.)"
					},
					{
						"name": "Sequence",
						"value": ">3W56:A|PDBID|CHAIN|SEQUENCE MRGSHHHHHHENLYFQGQNSHMQLRLYNLRVRGLPSDLMGITDGYVKVFCGSANLGETSVNHNNANPWWTEEFSHFKAQE NDILRLEVHDEDTFFDDLLGVCQRQIKVGTHEHDCYLKEGGTLHYMYTLSV"
					},
					{
						"name": "Space Group",
						"value": "I 1 2 1"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 25.35, b = 55.22, c = 89.75),(a = 90.00, b = 91.17, c = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://pdb.org/pdb/search/structidSearch.do?structureId=3W56"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "4297"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Identifier",
						"value": "http://www.ncbi.nlm.nih.gov/pubmed/24070258"
					},
					{
						"name": "Notes",
						"value": "Biochem J."
					},
					{
						"name": "Title",
						"value": "Defining the interaction of perforin with calcium and the phospholipid membrane."
					},
					{
						"name": "Type",
						"value": "Publication"
					}
				],
				"name": "Related Info"
			}
		],
		"id": 208,
		"size": 4152473600
	},
	{
		"datasets": [
			{
				"folder": "frames/Rohan",
				"file_count": 360,
				"description": "Raw data for C4_Zn_low_1",
				"size": 3020083200
			},
			{
				"folder": "frames/Rohan",
				"file_count": 1,
				"description": "Raw data for C4_Zn_low_0",
				"size": 8389120
			},
			{
				"folder": "frames/Rohan",
				"file_count": 360,
				"description": "Raw data for C4_Zn_high_1",
				"size": 3020083200
			},
			{
				"folder": "frames/Rohan",
				"file_count": 2,
				"description": "Raw data for C4_Zn_high_0",
				"size": 16778240
			},
			{
				"folder": "frames/Rohan",
				"file_count": 360,
				"description": "Raw data for C4_Ni_low_1",
				"size": 3020083200
			},
			{
				"folder": "frames/Rohan",
				"file_count": 4,
				"description": "Raw data for C4_Ni_low_0",
				"size": 33556480
			},
			{
				"folder": "frames/Rohan",
				"file_count": 360,
				"description": "Raw data for C4_Ni_high_1",
				"size": 3020083200
			},
			{
				"folder": "frames/Rohan",
				"file_count": 2,
				"description": "Raw data for C4_Ni_high_0",
				"size": 16778240
			},
			{
				"folder": "frames/Rohan",
				"file_count": 360,
				"description": "Raw data for C4_native_1",
				"size": 3020083200
			},
			{
				"folder": "frames/Rohan",
				"file_count": 2,
				"description": "Raw data for C4_native_0",
				"size": 16778240
			},
			{
				"folder": "frames",
				"file_count": 360,
				"description": "Raw data for B8_1",
				"size": 6794956800
			}
		],
		"name": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"date_modified": "2015-11-04 07:54:53.470903+00:00",
		"datafile_count": 2171,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "University of Western Australia",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 11,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Ni_low_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Ni_low_0"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Ni_high_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Ni_high_0"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Zn_low_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Zn_low_0"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Zn_high_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_Zn_high_0"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Dataset",
						"value": "Raw data for C4_native_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": ""
					},
					{
						"name": "Dataset",
						"value": "Raw data for C4_native_0"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "PDB ID",
						"value": "4MTT"
					},
					{
						"name": "Resolution",
						"value": "2.17\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2348"
					},
					{
						"name": "R-Value",
						"value": "0.1771(obs.)"
					},
					{
						"name": "Space Group",
						"value": "P 65 2 2"
					},
					{
						"name": "Title",
						"value": "Ni- and Zn-bound GloA2 at low resolution"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 51.070, b = 51.070, c = 327.630),(alpha = 90.00, beta = 90.00, gamma = 120.00)"
					},
					{
						"name": "URL",
						"value": "http://www.pdb.org/pdb/search/structidSearch.do?structureId=4MTT"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Native"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B8_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3375a"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX1"
					},
					{
						"name": "EPN",
						"value": "4207"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Acknowledgements",
						"value": "This research was undertaken on the MX1 and MX2 beamlines at the Australian Synchrotron, Victoria, Australia."
					},
					{
						"name": "DOI",
						"value": "10.4225/52/5639A2941B57F"
					}
				],
				"name": "Publication Details"
			},
			{
				"values": [
					{
						"name": "embargo",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Pseudomonas aeruginosa"
					},
					{
						"name": "Sequence",
						"value": "MRILHSMLRVADLEAALEFYTRALDMRLLRRRDYPEGRFTLAFVGYQDERAAAALELTHNWDRDGYTQGDGYGHLAIEVE\\nDAAVTCARARALGYRVTREAGLMQHGRSVIAFLEDPDGYKVELIQKGTQFD"
					}
				],
				"name": "Sequence Data"
			},
			{
				"values": [
					{
						"name": "pdb-last-sync",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Authors",
						"value": "Bythell-Douglas, R.; Suttisansanee, U.; Flematti, G.R.; Challenor, M.; Lee, M.; Panjikar, S.; Honek, J.F.; Bond, C.S."
					},
					{
						"name": "DOI",
						"value": "http://dx.doi.org/10.1002/chem.201405402"
					},
					{
						"name": "Journal",
						"value": "Chemistry"
					},
					{
						"name": "Pages",
						"value": "541-544"
					},
					{
						"name": "Title",
						"value": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity."
					},
					{
						"name": "Volume",
						"value": "21"
					}
				],
				"name": "Citation"
			}
		],
		"id": 1038,
		"size": 21987653120
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 0,
				"description": "Australian Synchrotron Proposal ID: 286",
				"size": 0
			}
		],
		"name": "Specialist Crystallography at the Australian Synchrotron",
		"date_modified": "2010-08-26 00:00:00+00:00",
		"datafile_count": 0,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 1,
		"authors": [
			"Jensen P",
			"Turner P"
		],
		"metadata": [],
		"id": 865,
		"size": 0
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 2,
				"description": "Ancillary Files",
				"size": 14446954
			},
			{
				"folder": null,
				"file_count": 127,
				"description": "Images",
				"size": 2397564928
			}
		],
		"name": "Bacterioferritin from Mycobacterium smegmatis contains zinc in its di-nuclear site.",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 129,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Janowski R",
			"Auerbach-Nevo T",
			"Weiss MS"
		],
		"metadata": [],
		"id": 876,
		"size": 2412011882
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 807,
				"description": "Log Files",
				"size": 1143534320
			},
			{
				"folder": null,
				"file_count": 360,
				"description": "Low Resolution",
				"size": 3020083200
			},
			{
				"folder": null,
				"file_count": 362,
				"description": "High Resolution",
				"size": 3020084172
			}
		],
		"name": "The X-ray crystal structure of Escherichia coli succinic semialdehyde dehydrogenase; structural insights into NADP+ / enzyme interactions",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 1529,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 3,
		"authors": [
			"Langendorf CG",
			"Key TLG",
			"Fenalti G",
			"Kan WT",
			"Buckle AM",
			"Caradoc-Davies T",
			"Tuck KL",
			"Law RHP",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 877,
		"size": 7183701692
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 720,
				"description": "xtal8 small oscillation angle",
				"size": 13589913600
			},
			{
				"folder": null,
				"file_count": 3437,
				"description": "xtal8 small oscillation angle",
				"size": 28826331136
			}
		],
		"name": "Structure of the Plasmodium falciparum PfA-M17",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 4157,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan"
		],
		"metadata": [],
		"id": 881,
		"size": 42416244736
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 1,
				"description": "Ancillary Files",
				"size": 261880
			},
			{
				"folder": null,
				"file_count": 298,
				"description": "Native",
				"size": 6645375372
			}
		],
		"name": "X-ray crystal structure of MENT: evidence for functional loop-sheet polymers in chromatin condensation.",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 299,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Irving JA",
			"McGowan S",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 889,
		"size": 6645637252
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 9,
				"description": "M1_3EBG Structure of the M1 Alanylaminopeptidase from malaria<br/>9_refmac5_final.log Final refinement log file. Refmac v5.2.0019<br/>80_scala.log Processing log file from SCALA<br/>p212121_scala1.mtz Processed data<br/><br/>M1_BES_3EBH Structure of the M1 Alanylaminopeptidase from malaria complexed with bestatin <br/>4_refmac5_finalBES.log Final refinement log file. Refmac v5.2.0019<br/>41_scala.log Processing log file from SCALA<br/>xtal8c_p212121_scala1.mtz Processed data<br/><br/>M1_Co4_3EBI Structure of the M1 Alanylaminopeptidase from malaria complexed with the phosphinate dipeptide analog<br/>.osc files Raw image files. In-house Rotating Anode (Rigaku RAxis IV)<br/>12_refmac5_final Co4.log Final refinement log file. Refmac v5.2.0019<br/>45_scala.log Processing log file from SCALA<br/>3EBIco4_p212121_scala1.mtz Processed data",
				"size": 24868300
			},
			{
				"folder": null,
				"file_count": 470,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria complexed with bestatin",
				"size": 8414258830
			},
			{
				"folder": null,
				"file_count": 363,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria complexed with the phosphinate dipeptide analog",
				"size": 3029308012
			},
			{
				"folder": null,
				"file_count": 275,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria",
				"size": 4907818658
			}
		],
		"name": "Structural basis for the inhibition of the essential Plasmodium falciparum M1 neutral aminopeptidase",
		"date_modified": "2010-04-30 00:00:00+00:00",
		"datafile_count": 1117,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 4,
		"authors": [
			"Buckle AM",
			"Porter CJ",
			"Whisstock JC",
			"McGowan S"
		],
		"metadata": [],
		"id": 893,
		"size": 16376253800
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 5,
				"description": "AprV2.C141S",
				"size": 23404595
			},
			{
				"folder": null,
				"file_count": 5,
				"description": "AprB2",
				"size": 44153579
			},
			{
				"folder": null,
				"file_count": 5,
				"description": "AprV2",
				"size": 20377941
			},
			{
				"folder": null,
				"file_count": 364,
				"description": "AprV2.C141S",
				"size": 6300308888
			},
			{
				"folder": null,
				"file_count": 799,
				"description": "AprB2",
				"size": 14067553216
			},
			{
				"folder": null,
				"file_count": 446,
				"description": "AprV2",
				"size": 7747352085
			}
		],
		"name": "Virulence-associated subtilisin-like proteases that use a novel disulphide-tethered exosite to mediate substrate specificity",
		"date_modified": "2010-11-25 00:00:00+00:00",
		"datafile_count": 1624,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 6,
		"authors": [
			"Kennan RM",
			"Wong W",
			"Dhungyel O",
			"Han X",
			"Wong D",
			"Parker D",
			"Rosado CJ",
			"Law RHP",
			"McGowan S",
			"Reeve SB",
			"Levina V",
			"Powers GA",
			"Pike RN",
			"Bottomley SP",
			"Smith AI",
			"Marsh I",
			"Whittington RJ",
			"Whisstock JC",
			"Porter CJ",
			"Rood JI"
		],
		"metadata": [],
		"id": 894,
		"size": 28203150304
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 753,
				"description": "3T8W_Dataset2",
				"size": 14212784640
			},
			{
				"folder": null,
				"file_count": 999,
				"description": "3T8W_dataset1",
				"size": 18856005120
			}
		],
		"name": "A bestatin-based chemical biology strategy reveals distinct roles for malaria M1- and M17-family aminopeptidases",
		"date_modified": "2011-09-23 00:00:00+00:00",
		"datafile_count": 1752,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"McGowan S",
			"Klemba M",
			"Greenbaum DC"
		],
		"metadata": [],
		"id": 897,
		"size": 33068789760
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 241,
				"description": "Images",
				"size": 4288573765
			}
		],
		"name": "X-ray crystal structure of the fibrinolysis inhibitor {alpha}2-antiplasmin",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 241,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 1,
		"authors": [
			"Law RHP",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 890,
		"size": 4288573765
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 2,
				"description": "Ancillary Files",
				"size": 60295696
			},
			{
				"folder": null,
				"file_count": 665,
				"description": "Inhouse",
				"size": 11708139157
			}
		],
		"name": "The 1.6 A Crystal Structure of the Catalytic Domain of PlyB, a Bacteriophage Lysin Active Against Bacillus anthracis.",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 667,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Porter CJ",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 885,
		"size": 11768434853
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 5,
				"description": "Ancillary Files",
				"size": 174248890
			},
			{
				"folder": null,
				"file_count": 215,
				"description": "Native 2.0 Angstroms Collection",
				"size": 1795314692
			},
			{
				"folder": null,
				"file_count": 388,
				"description": "Derivative (KI) in-house",
				"size": 6950897572
			},
			{
				"folder": null,
				"file_count": 688,
				"description": "Derivative (KPtCl) inhouse Collection date",
				"size": 12370128148
			},
			{
				"folder": null,
				"file_count": 469,
				"description": "Derivative (EMSA) in-house",
				"size": 8426851012
			},
			{
				"folder": null,
				"file_count": 511,
				"description": "Native 2.5 Angstroms Collection",
				"size": 9166245940
			}
		],
		"name": "A common fold mediates vertebrate defense and bacterial attack",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2276,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 6,
		"authors": [
			"Buckle AM",
			"Dunstone MA",
			"Law RHP",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 878,
		"size": 38883686254
	},
	{
		"datasets": [
			{
				"folder": "frames",
				"file_count": 360,
				"description": "Raw data for D3_2",
				"size": 6794956800
			},
			{
				"folder": "frames",
				"file_count": 360,
				"description": "Raw data for D3_1",
				"size": 6794956800
			}
		],
		"name": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"date_modified": "2015-11-05 22:14:06.262874+00:00",
		"datafile_count": 720,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "University of Western Australia",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 2,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"metadata": [
			{
				"values": [],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "PDB ID",
						"value": "4MTS"
					},
					{
						"name": "Resolution",
						"value": "1.8\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2006"
					},
					{
						"name": "R-Value",
						"value": "0.1655(obs.)"
					},
					{
						"name": "Space Group",
						"value": "P 65 2 2"
					},
					{
						"name": "Title",
						"value": "Ni- and Zn-bound GloA2 at high resolution"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 50.700, b = 50.700, c = 327.640),(alpha = 90.00, beta = 90.00, gamma = 120.00)"
					},
					{
						"name": "URL",
						"value": "http://www.pdb.org/pdb/search/structidSearch.do?structureId=4MTS"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Native"
					},
					{
						"name": "Dataset",
						"value": "Raw data for D3_2"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Native"
					},
					{
						"name": "Dataset",
						"value": "Raw data for D3_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3375a"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Acknowledgements",
						"value": "This research was undertaken on the MX2 beamline at the Australian Synchrotron, Victoria, Australia."
					},
					{
						"name": "DOI",
						"value": "10.4225/52/563BD4A36013F"
					}
				],
				"name": "Publication Details"
			},
			{
				"values": [
					{
						"name": "embargo",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Pseudomonas aeruginosa"
					},
					{
						"name": "Sequence",
						"value": "MRILHSMLRVADLEAALEFYTRALDMRLLRRRDYPEGRFTLAFVGYQDERAAAALELTHNWDRDGYTQGDGYGHLAIEVE\\nDAAVTCARARALGYRVTREAGLMQHGRSVIAFLEDPDGYKVELIQKGTQFD"
					}
				],
				"name": "Sequence Data"
			},
			{
				"values": [
					{
						"name": "pdb-last-sync",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Authors",
						"value": "Bythell-Douglas, R.; Suttisansanee, U.; Flematti, G.R.; Challenor, M.; Lee, M.; Panjikar, S.; Honek, J.F.; Bond, C.S."
					},
					{
						"name": "DOI",
						"value": "http://dx.doi.org/10.1002/chem.201405402"
					},
					{
						"name": "Journal",
						"value": "Chemistry"
					},
					{
						"name": "Pages",
						"value": "541-544"
					},
					{
						"name": "Title",
						"value": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity."
					},
					{
						"name": "Volume",
						"value": "21"
					}
				],
				"name": "Citation"
			}
		],
		"id": 1037,
		"size": 13589913600
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 1,
				"description": "Ancillary Files",
				"size": 1806952
			},
			{
				"folder": null,
				"file_count": 515,
				"description": "Native",
				"size": 9184646875
			}
		],
		"name": "The crystal structure of DehI reveals a new alpha-haloacid dehalogenase fold and active-site mechanism.",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 516,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Wilce JW",
			"Schmidberger MCJ"
		],
		"metadata": [],
		"id": 886,
		"size": 9186453827
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 0,
				"description": "Ancillary Files",
				"size": 0
			},
			{
				"folder": null,
				"file_count": 177,
				"description": "APS Source 80 Deg 5 Sec 0.5 Deg - p212121",
				"size": 1742775079
			},
			{
				"folder": null,
				"file_count": 279,
				"description": "APS Source 125 Deg (includes xfiles, denzo)",
				"size": 2857787943
			}
		],
		"name": "The high resolution crystal structure of a native thermostable serpin reveals the complex mechanism underpinning the stressed to relaxed transition.",
		"date_modified": "2011-09-12 00:00:00+00:00",
		"datafile_count": 456,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 3,
		"authors": [
			"Bottomley SP",
			"Buckle AM",
			"Butcher RE",
			"Cabrita LD",
			"Fulton KF",
			"Irving JA",
			"Lesk AM",
			"Reeve SL",
			"Rossjohn J",
			"Smith I",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 896,
		"size": 4600563022
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 132,
				"description": "Native",
				"size": 2269517049
			}
		],
		"name": "The structure of granzyme C reveals an unusual mechanism of protease auto-inhibition",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 132,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 1,
		"authors": [
			"Buckle AM",
			"Kaiserman D",
			"Law RHP",
			"Whisstock JC",
			"Bird PI"
		],
		"metadata": [],
		"id": 888,
		"size": 2269517049
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 5,
				"description": "Ancillary Files",
				"size": 90765842
			},
			{
				"folder": null,
				"file_count": 121,
				"description": "Native 2.0 Angstroms",
				"size": 2160726148
			}
		],
		"name": "Crystal structure of scabies mite inactivated protease paralogue S-D1 (SMIPP-S-D1)",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 126,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Buckle AM"
		],
		"metadata": [],
		"id": 879,
		"size": 2251491990
	},
	{
		"datasets": [
			{
				"folder": "frames/B5",
				"file_count": 360,
				"description": "Raw data for B5_2",
				"size": 6794956800
			},
			{
				"folder": "frames/B5",
				"file_count": 360,
				"description": "Raw data for B5_1",
				"size": 6794956800
			},
			{
				"folder": "frames/F6",
				"file_count": 180,
				"description": "Raw data for F6_1",
				"size": 3397478400
			}
		],
		"name": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"date_modified": "2015-10-28 03:36:10.728308+00:00",
		"datafile_count": 900,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "University of Western Australia",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 3,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"metadata": [
			{
				"values": [],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "PDB ID",
						"value": "4MTR"
					},
					{
						"name": "Resolution",
						"value": "1.83\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2187"
					},
					{
						"name": "R-Value",
						"value": "0.1836(obs.)"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Title",
						"value": "Zn-bound GloA2"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 40.788, b = 73.503, c = 80.011),(alpha = 90.00, beta = 90.00, gamma = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.pdb.org/pdb/search/structidSearch.do?structureId=4MTR"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Native"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B5_2"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Zn Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B5_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Zn Anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for F6_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3051"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "2680"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Acknowledgements",
						"value": "This research was undertaken on the MX2 beamline at the Australian Synchrotron, Victoria, Australia."
					},
					{
						"name": "DOI",
						"value": "10.4225/52/5636F4F2488F0"
					}
				],
				"name": "Publication Details"
			},
			{
				"values": [
					{
						"name": "embargo",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Pseudomonas aeruginosa"
					},
					{
						"name": "Sequence",
						"value": "MRILHSMLRVADLEAALEFYTRALDMRLLRRRDYPEGRFTLAFVGYQDERAAAALELTHNWDRDGYTQGDGYGHLAIEVE\\nDAAVTCARARALGYRVTREAGLMQHGRSVIAFLEDPDGYKVELIQKGTQFD"
					}
				],
				"name": "Sequence Data"
			},
			{
				"values": [
					{
						"name": "pdb-last-sync",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Authors",
						"value": "Bythell-Douglas, R.; Suttisansanee, U.; Flematti, G.R.; Challenor, M.; Lee, M.; Panjikar, S.; Honek, J.F.; Bond, C.S."
					},
					{
						"name": "DOI",
						"value": "http://dx.doi.org/10.1002/chem.201405402"
					},
					{
						"name": "Journal",
						"value": "Chemistry"
					},
					{
						"name": "Pages",
						"value": "541-544"
					},
					{
						"name": "Title",
						"value": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity."
					},
					{
						"name": "Volume",
						"value": "21"
					}
				],
				"name": "Citation"
			}
		],
		"id": 1035,
		"size": 16987392000
	},
	{
		"datasets": [
			{
				"folder": "frames",
				"file_count": 360,
				"description": "Raw data for C7_Znlow_1",
				"size": 6794956800
			},
			{
				"folder": "frames",
				"file_count": 360,
				"description": "Raw data for C7_Znhigh_1",
				"size": 6794956800
			},
			{
				"folder": "frames/B7",
				"file_count": 360,
				"description": "Raw data for B7_3",
				"size": 6794956800
			},
			{
				"folder": "frames/B7",
				"file_count": 360,
				"description": "Raw data for B7_2",
				"size": 6794956800
			},
			{
				"folder": "frames/B7",
				"file_count": 360,
				"description": "Raw data for B7_1",
				"size": 6794956800
			}
		],
		"name": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"date_modified": "2015-11-05 22:13:07.722484+00:00",
		"datafile_count": 1800,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "University of Western Australia",
			"license": {
				"url": "http://creativecommons.org/licenses/by/3.0/au/",
				"image_url": "http://i.creativecommons.org/l/by/3.0/au/88x31.png",
				"license": "Creative Commons Attribution 3.0 Australia (CC BY 3.0)"
			}
		},
		"dataset_count": 5,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"metadata": [
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Native"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B7_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Zn anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C7_Znlow_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Zn anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for C7_Znhigh_1"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Ni anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B7_3"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [
					{
						"name": "Additional information",
						"value": "Ni anomalous"
					},
					{
						"name": "Dataset",
						"value": "Raw data for B7_2"
					}
				],
				"name": "MX dataset details"
			},
			{
				"values": [],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "PDB ID",
						"value": "4MTQ"
					},
					{
						"name": "Resolution",
						"value": "2.17\u00c5"
					},
					{
						"name": "R-Free",
						"value": "0.2329"
					},
					{
						"name": "R-Value",
						"value": "0.1853(obs.)"
					},
					{
						"name": "Space Group",
						"value": "P 21 21 21"
					},
					{
						"name": "Title",
						"value": "Ni-bound GloA2"
					},
					{
						"name": "Unit Cell (\u00c5,\u00b0)",
						"value": "(a = 41.145, b = 74.983, c = 80.362),(alpha = 90.00, beta = 90.00, gamma = 90.00)"
					},
					{
						"name": "URL",
						"value": "http://www.pdb.org/pdb/search/structidSearch.do?structureId=4MTQ"
					}
				],
				"name": "Protein Data Bank"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3051"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Beamline",
						"value": "MX2"
					},
					{
						"name": "EPN",
						"value": "3375a"
					}
				],
				"name": "Experiment ID"
			},
			{
				"values": [
					{
						"name": "Acknowledgements",
						"value": "This research was undertaken on the MX2 beamline at the Australian Synchrotron, Victoria, Australia."
					},
					{
						"name": "DOI",
						"value": "10.4225/52/563BD46765F73"
					}
				],
				"name": "Publication Details"
			},
			{
				"values": [
					{
						"name": "embargo",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Expression System",
						"value": "Escherichia coli"
					},
					{
						"name": "Organism",
						"value": "Pseudomonas aeruginosa"
					},
					{
						"name": "Sequence",
						"value": "MRILHSMLRVADLEAALEFYTRALDMRLLRRRDYPEGRFTLAFVGYQDERAAAALELTHNWDRDGYTQGDGYGHLAIEVE\\nDAAVTCARARALGYRVTREAGLMQHGRSVIAFLEDPDGYKVELIQKGTQFD"
					}
				],
				"name": "Sequence Data"
			},
			{
				"values": [
					{
						"name": "pdb-last-sync",
						"value": null
					}
				],
				"name": "Publication"
			},
			{
				"values": [
					{
						"name": "Authors",
						"value": "Bythell-Douglas, R.; Suttisansanee, U.; Flematti, G.R.; Challenor, M.; Lee, M.; Panjikar, S.; Honek, J.F.; Bond, C.S."
					},
					{
						"name": "DOI",
						"value": "http://dx.doi.org/10.1002/chem.201405402"
					},
					{
						"name": "Journal",
						"value": "Chemistry"
					},
					{
						"name": "Pages",
						"value": "541-544"
					},
					{
						"name": "Title",
						"value": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity."
					},
					{
						"name": "Volume",
						"value": "21"
					}
				],
				"name": "Citation"
			}
		],
		"id": 1036,
		"size": 33974784000
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 801,
				"description": "zinc soak unbound",
				"size": 15118778880
			},
			{
				"folder": null,
				"file_count": 999,
				"description": "zinc soak unbound",
				"size": 18856005120
			}
		],
		"name": "Structure of the Plasmodium falciparum PfA-M17_ZnZn",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 1800,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan"
		],
		"metadata": [],
		"id": 884,
		"size": 33974784000
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 2,
				"description": "Ancillary Files",
				"size": 15199234
			},
			{
				"folder": null,
				"file_count": 184,
				"description": "Inhouse",
				"size": 3244250373
			}
		],
		"name": "The Crystal Structure of the Transthyretin-like Protein from Salmonella dublin, a Prokaryote 5-Hydroxyisourate Hydrolase",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 186,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Law RHP",
			"Whisstock JC"
		],
		"metadata": [],
		"id": 887,
		"size": 3259449607
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 755,
				"description": "PfA-M17 with Compound4 bound in active sites",
				"size": 14250534400
			},
			{
				"folder": null,
				"file_count": 999,
				"description": "PfA-M17 with Compound4 bound in active sites",
				"size": 18856005120
			},
			{
				"folder": null,
				"file_count": 686,
				"description": "PfA-M17 with Compound4 bound in active sites",
				"size": 13434234622
			},
			{
				"folder": null,
				"file_count": 302,
				"description": "PfA-M17 with Compound 4 bound in active site",
				"size": 5883232226
			}
		],
		"name": "Structure of the Plasmodium falciparum PfA-M17_Co4",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2742,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 4,
		"authors": [
			"Sheena McGowan"
		],
		"metadata": [],
		"id": 883,
		"size": 52424006368
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 994,
				"description": "mouse_perforin_R213E_I",
				"size": 18761630720
			},
			{
				"folder": null,
				"file_count": 1114,
				"description": "mouse_perforin_R213E_Ir",
				"size": 21026616320
			},
			{
				"folder": null,
				"file_count": 1338,
				"description": "mouse_perforin_R213E_Hg",
				"size": 25254589440
			},
			{
				"folder": null,
				"file_count": 947,
				"description": "mouse_perforin_R213E_Native2",
				"size": 17874511360
			},
			{
				"folder": null,
				"file_count": 736,
				"description": "mouse_perforin_R213E_Native1",
				"size": 13891911680
			},
			{
				"folder": null,
				"file_count": 10,
				"description": "Scala Logs & BUSTER Output",
				"size": 10303319
			}
		],
		"name": "The structural basis for membrane binding and pore formation by lymphocyte perforin",
		"date_modified": "2011-05-27 00:00:00+00:00",
		"datafile_count": 5139,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 6,
		"authors": [
			"Ruby Law",
			"Natalya Lukoyanova",
			"Ilia Voskoboinik",
			"Tom Caradoc-Davies",
			"Katherine Baran",
			"Michelle A. Dunstone",
			"Michael E. D\u2019Angelo",
			"Fasseli Coulibaly",
			"Sandra Verschoor",
			"Kylie A. Browne",
			"Annette Ciccone",
			"Michael J. Kuiper",
			"Phillip I. Bird",
			"Joseph A. Trapani",
			"Helen R. Saibil",
			"James C. Whisstock"
		],
		"metadata": [],
		"id": 895,
		"size": 96819562839
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 2644,
				"description": "PfA-M17 with Bestatin bound in active site",
				"size": 22172450308
			}
		],
		"name": "Structure of the Plasmodium falciparum PfA-M17_BES",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2644,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 1,
		"authors": [
			"Sheena McGowan"
		],
		"metadata": [],
		"id": 882,
		"size": 22172450308
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 1,
				"description": "Ancillary Files",
				"size": 2290824
			},
			{
				"folder": null,
				"file_count": 201,
				"description": "Native",
				"size": 1249911176
			}
		],
		"name": "Crystal structures of the substrate free-enzyme, and reaction intermediate of the HAD superfamily member, haloacid dehalogenase DehIVa from Burkholderia cepacia MBA4.",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 202,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Schmidberger MCJ",
			"Wilce MCJ"
		],
		"metadata": [],
		"id": 880,
		"size": 1252202000
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 252,
				"description": "Glycoform 1 Humanplasminogen (4DUU)",
				"size": 2390143824
			},
			{
				"folder": null,
				"file_count": 320,
				"description": "Glycoform 2 Humanplasminogen (4DUR)",
				"size": 3052835356
			}
		],
		"name": "The X-ray Crystal Structure of Full-length Human Plasminogen",
		"date_modified": "2012-08-22 00:00:00+00:00",
		"datafile_count": 572,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {
				"url": "http://creativecommons.org/licenses/by-nc-nd/2.5/au/",
				"image_url": "https://i.creativecommons.org/l/by-nc-nd/2.5/au/88x31.png",
				"license": "Creative Commons Attribution-NonCommercial-NoDerivs 2.5 Australia (CC BY-NC-ND 2.5 AU)"
			}
		},
		"dataset_count": 2,
		"authors": [
			"Ruby H.P. Law",
			"Tom Caradoc-Davies",
			"Nathan Cowieson",
			"Anita J. Horvath",
			"Adam J. Quek",
			"Joanna Amarante Encarnacao",
			"David Steer",
			"Angus Cowan",
			"Qingwei Zhang",
			"Bernadine G.C. Lu",
			"Robert N. Pike",
			"A. Ian Smith",
			"Paul B. Coughlin",
			"James C. Whisstock"
		],
		"metadata": [],
		"id": 899,
		"size": 5442979180
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 240,
				"description": "M18",
				"size": 2286661748
			}
		],
		"name": "X-ray crystal structure and specificity of the Plasmodium falciparum malaria aminopeptidase PfM18AAP",
		"date_modified": "2012-08-09 00:00:00+00:00",
		"datafile_count": 240,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {
				"url": "http://creativecommons.org/licenses/by-nc/2.5/au/",
				"image_url": "http://i.creativecommons.org/l/by-nc/2.5/au/88x31.png",
				"license": "Creative Commons Attribution-NonCommercial 2.5 Australia (CC BY-NC 2.5 AU)"
			}
		},
		"dataset_count": 1,
		"authors": [
			"Sheena McGowan"
		],
		"metadata": [],
		"id": 900,
		"size": 2286661748
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 411,
				"description": "PlyC",
				"size": 6990462420
			},
			{
				"folder": null,
				"file_count": 120,
				"description": "PlyCB",
				"size": 4026593280
			}
		],
		"name": "X-ray crystal structure of the streptococcal specific phage lysin PlyC",
		"date_modified": "2012-08-22 00:00:00+00:00",
		"datafile_count": 531,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {
				"url": "http://creativecommons.org/licenses/by-nc/2.5/au/",
				"image_url": "http://i.creativecommons.org/l/by-nc/2.5/au/88x31.png",
				"license": "Creative Commons Attribution-NonCommercial 2.5 Australia (CC BY-NC 2.5 AU)"
			}
		},
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan",
			"Ashley Buckle",
			"James Whisstock"
		],
		"metadata": [],
		"id": 901,
		"size": 11017055700
	},
	{
		"datasets": [
			{
				"folder": null,
				"file_count": 720,
				"description": "Raw data for Compound 7c",
				"size": 6040166400
			},
			{
				"folder": null,
				"file_count": 720,
				"description": "Raw data for Compound 7d",
				"size": 13589913600
			}
		],
		"name": "Synthesis of new (-)-Bestatin-based inhibitor libraries reveals a novel binding mode in the S1 pocket of the essential malaria M1 metalloaminopeptidase.",
		"date_modified": "2012-11-15 00:00:00+00:00",
		"datafile_count": 1440,
		"description": {
			"admins": [
				"ulrich.felzmann"
			],
			"institution": "Monash University",
			"license": {}
		},
		"dataset_count": 2,
		"authors": [
			"Geetha Velmourougane",
			"Michael B. Harbut",
			"Seema Dalal",
			"Sheena McGowan",
			"Christine A. Oellig",
			"James C. Whisstock",
			"Michael Klemba",
			"Doron C. Greenbaum"
		],
		"metadata": [],
		"id": 898,
		"size": 19630080000
	}
]
const exp_array = [
	{
		"datasets": [
			{
				"id": 11662,
				"description": "H4_1"
			}
		],
		"description": "Streptococcus pyogenes (group A Streptococcus [GAS]) causes ~700 million human infections/year, resulting in >500,000 deaths. There is no commercial GAS vaccine available. The GAS surface protein arginine deiminase (ADI) protects mice against a lethal challenge. ADI is an enzyme that converts arginine to citrulline and ammonia. Administration of a GAS vaccine preparation containing wild-type ADI, a protein with inherent enzymatic activity, may present a safety risk. In an approach intended to maximize the vaccine safety of GAS ADI, X-ray crystallography and structural immunogenic epitope mapping were used to inform vaccine design. This study aimed to knock out ADI enzyme activity without disrupting the three-dimensional structure or the recognition of immunogenic epitopes. We determined the crystal structure of ADI at 2.5 \\u00c5 resolution and used it to select a number of amino acid residues for mutagenesis to alanine (D166, E220, H275, D277, and C401). Each mutant protein displayed abrogated activity, and three of the mutant proteins (those with the D166A, H275A, and D277A mutations) possessed a secondary structure and oligomerization state equivalent to those of the wild type, produced high-titer antisera, and avoided disruption of B-cell epitopes of ADI. In addition, antisera raised against the D166A and D277A mutant proteins bound to the GAS cell surface. The inactivated D166A and D277A mutant ADIs are ideal for inclusion in a GAS vaccine preparation. There is no human ortholog of ADI, and we confirm that despite limited structural similarity in the active-site region to human peptidyl ADI 4 (PAD4), ADI does not functionally mimic PAD4 and antiserum raised against GAS ADI does not recognize human PAD4.",
		"date_modified": "2014-03-24 05:28:04.911373+00:00",
		"datafile_count": 720,
		"title": "Structure-Informed Design of an Enzymatically Inactive Vaccine Component for Group A Streptococcus",
		"dataset_count": 1,
		"authors": [
			"Anna Henningham",
			"Daniel J. Ericsson",
			"Karla Langer",
			"Lachlan Casey",
			"Blagojce Jovcevski",
			"G. Singh Chhatwal",
			"J. Andrew Aquilina",
			"Michael R. Batzloff",
			"Bostjan Kobe",
			"Mark Walker"
		],
		"id": 222
	},
	{
		"datasets": [
			{
				"id": 13355,
				"description": "D6_2"
			},
			{
				"id": 13354,
				"description": "D6_1"
			},
			{
				"id": 13353,
				"description": "D6_0"
			},
			{
				"id": 13352,
				"description": "C6_1"
			},
			{
				"id": 13351,
				"description": "C5_1"
			},
			{
				"id": 13350,
				"description": "B7_2"
			},
			{
				"id": 13349,
				"description": "B6_2"
			},
			{
				"id": 13347,
				"description": "A8_2"
			},
			{
				"id": 13345,
				"description": "A6_2"
			},
			{
				"id": 13344,
				"description": "A5_2"
			}
		],
		"description": "Aldo-keto reductase 1C3 (AKR1C3) catalyses the NADPH dependent reduction of carbonyl groups in a number of important steroid and prostanoid molecules. The enzyme is also over-expressed in prostate and breast cancer and its expression is correlated with the aggressiveness of the disease. The steroid products of AKR1C3 catalysis are important in proliferative signalling of hormone-responsive cells, while the prostanoid products promote prostaglandin-dependent proliferative pathways. In these ways, AKR1C3 contributes to tumour development and maintenance, and suggest that inhibition of AKR1C3 activity is an attractive target for the development of new anti-cancer therapies. Non-steroidal anti-inflammatory drugs (NSAIDs) are one well-known class of compounds that inhibits AKR1C3, yet crystal structures have only been determined for this enzyme with flufenamic acid, indomethacin, and closely related analogues bound. While the flufenamic acid and indomethacin structures have been used to design novel inhibitors, they provide only limited coverage of the NSAIDs that inhibit AKR1C3 and that may be used for the development of new AKR1C3 targeted drugs. To understand how other NSAIDs bind to AKR1C3, we have determined ten crystal structures of AKR1C3 complexes that cover three different classes of NSAID, N-phenylanthranilic acids (meclofenamic acid, mefenamic acid), arylpropionic acids (flurbiprofen, ibuprofen, naproxen), and indomethacin analogues (indomethacin, sulindac, zomepirac). The N-phenylanthranilic and arylpropionic acids bind to common sites including the enzyme catalytic centre and a constitutive active site pocket, with the arylpropionic acids probing the constitutive pocket more effectively. By contrast, indomethacin and the indomethacin analogues sulindac and zomepirac, display three distinctly different binding modes that explain their relative inhibition of the AKR1C family members. This new data from ten crystal structures greatly broadens the base of structures available for future structure-guided drug discovery efforts.\\r\\n\\r\\nThis work was funded by Lottery Health Research (CJS; grant number 265027), the Auckland Medical Research Foundation (JUF and CJS; grant number 1110004. JUF; grant number 1109008), the National eScience Infrastructure (JUF), and the Maurice Wilkins Centre for Molecular Biodiscovery Flexible Research Seeding Programme (JUF and CJS). We further acknowledge salary support from the Maurice Wilkins Centre for Molecular Biodiscovery (CJS, JUF) and Summer Studentship funding from the Faculty of Science, University of Auckland (CJS, RMT).",
		"date_modified": "2014-03-24 05:27:55.791952+00:00",
		"datafile_count": 1615,
		"title": "Crystal Structures of Three Classes of Non-Steroidal Anti-Inflammatory Drugs in Complex with Aldo-Keto Reductase 1C3",
		"dataset_count": 10,
		"authors": [
			"Yuliana Yosaatmadja",
			"Jack Flanagan"
		],
		"id": 251
	},
	{
		"datasets": [
			{
				"id": 13363,
				"description": "G6_3"
			},
			{
				"id": 13361,
				"description": "G6_2"
			},
			{
				"id": 13360,
				"description": "G6_1"
			},
			{
				"id": 13357,
				"description": "D6_1"
			}
		],
		"description": "Proteins of the Drosophila behavior/human splicing (DBHS) family include mammalian SFPQ (PSF), NONO (p54nrb), PSPC1, and invertebrate NONA and Hrp65. DBHS proteins are predominately nuclear, and are involved in transcriptional and posttranscriptional gene regulatory functions as well as DNA repair. DBHS proteins influence a wide gamut of biological processes, including the regulation of circadian rhythm, carcinogenesis, and progression of cancer. Additionally, mammalian DBHS proteins associate with the architectural long noncoding RNA NEAT1 (Men\\u03b5/\\u03b2) to form paraspeckles, subnuclear bodies that alter gene expression via the nuclear retention of RNA. Here we describe the crystal structure of the heterodimer of the multidomain conserved region of the DBHS proteins, PSPC1 and NONO. These proteins form an extensively intertwined dimer, consistent with the observation that the different DBHS proteins are typically copurified from mammalian cells, and suggesting that they act as obligate heterodimers. The PSPC1/NONO heterodimer has a right-handed antiparallel coiled-coil that positions two of four RNA recognition motif domains in an unprecedented arrangement on either side of a 20-\\u00c5 channel. This configuration is supported by a protein:protein interaction involving the NONA/paraspeckle domain, which is characteristic of the DBHS family. By examining various mutants and truncations in cell culture, we find that DBHS proteins require an additional antiparallel coiled-coil emanating from either end of the dimer for paraspeckle subnuclear body formation. These results suggest that paraspeckles may potentially form through self-association of DBHS dimers into higher-order structures.",
		"date_modified": "2014-03-24 05:27:47.031404+00:00",
		"datafile_count": 2880,
		"title": "Structure of the heterodimer of human NONO and paraspeckle protein component 1",
		"dataset_count": 4,
		"authors": [
			"Charlie Bond ",
			"Daniel Passon ",
			"Mihwa Lee "
		],
		"id": 257
	},
	{
		"datasets": [
			{
				"id": 11373,
				"description": "2K3e_5"
			},
			{
				"id": 11372,
				"description": "2K3d_4"
			},
			{
				"id": 11371,
				"description": "2K7c_3"
			},
			{
				"id": 11370,
				"description": "2K3c_3"
			}
		],
		"description": "Rotary ATPases couple ATP hydrolysis/synthesis with proton translocation across biological membranes and so are central components of the biological energy conversion machinery. Their peripheral stalks are essential components that counteract torque generated by rotation of the central stalk during ATP synthesis or hydrolysis.\\r\\n\\r\\nThese datasets are derivatives of the peripheral stalk from T.thermophilus A-ATPase. Native crystals were soaked in Lutetium(III) acetate (2K7c_3_###.img) and Dysprosium(III) chloride (2K3#######.img).\\r\\n\\r\\nResulting maps were used to create the pdb model 3V6I. The model was used to identify bending and twisting motions inherent within the structure that accommodate movements within the ATPase.",
		"date_modified": "2014-03-24 05:28:13.456665+00:00",
		"datafile_count": 575,
		"title": "Derivatives for structure solution of the peripheral stalk from T.thermophilus A-ATPase",
		"dataset_count": 4,
		"authors": [
			"Alastair Stewart",
			"Daniela Stock"
		],
		"id": 210
	},
	{
		"datasets": [
			{
				"id": 9492,
				"description": "testcrystal_0"
			},
			{
				"id": 9491,
				"description": "testcrystal_1"
			}
		],
		"description": "This is a test experiment.",
		"date_modified": "2014-03-24 05:28:39.618975+00:00",
		"datafile_count": 181,
		"title": "Test Experiment MX1cal20130930",
		"dataset_count": 2,
		"authors": [],
		"id": 186
	},
	{
		"datasets": [
			{
				"id": 11374,
				"description": "smax_mg_D4_3"
			}
		],
		"description": "Following its secretion from cytotoxic lymphocytes into the immune synapse, perforin binds to target cell membranes through its Ca2 + -dependent C2 domain. Membrane-bound perforin then forms pores that allow passage of pro-apoptopic granzymes into the target cell. In the present study, structural and biochemical studiesrevealthatCa2+ bindingtriggersaconformationalchange in the C2 domain that permits four key hydrophobic residues to interact with the plasma membrane. However, in contrast with previous suggestions, these movements and membrane binding do not trigger irreversible conformational changes in the pore-forming MACPF (membrane attack complex/perforin- like) domain, indicating that subsequent monomer\\u2013monomer interactions at the membrane surface are required for perforin pore formation.\\r\\n\\r\\nPublication: Biochem J.",
		"date_modified": "2014-03-25 00:39:58.705267+00:00",
		"datafile_count": 220,
		"title": "Defining the interaction of perforin with calcium and the phospholipid membrane",
		"dataset_count": 1,
		"authors": [
			"Daouda A.K. Traore",
			"James C Whisstock"
		],
		"id": 208
	},
	{
		"datasets": [
			{
				"id": 54431,
				"description": "Raw data for C4_Zn_low_1"
			},
			{
				"id": 54430,
				"description": "Raw data for C4_Zn_low_0"
			},
			{
				"id": 54429,
				"description": "Raw data for C4_Zn_high_1"
			},
			{
				"id": 54428,
				"description": "Raw data for C4_Zn_high_0"
			},
			{
				"id": 54427,
				"description": "Raw data for C4_Ni_low_1"
			},
			{
				"id": 54426,
				"description": "Raw data for C4_Ni_low_0"
			},
			{
				"id": 54425,
				"description": "Raw data for C4_Ni_high_1"
			},
			{
				"id": 54424,
				"description": "Raw data for C4_Ni_high_0"
			},
			{
				"id": 54423,
				"description": "Raw data for C4_native_1"
			},
			{
				"id": 54422,
				"description": "Raw data for C4_native_0"
			},
			{
				"id": 54320,
				"description": "Raw data for B8_1"
			}
		],
		"description": "The Zn inactive class of glyoxalase I (Glo1) enzymes are metalloenzymes that are typically homodimeric with two metal-dependent active sites. While the two active sites share identical amino acid composition, this class of enzyme is optimally active with only one metal per homodimer. We have determined the X-ray crystal structure of GloA2, one of the Zn inactive Glo1 enzymes from Pseudomonas aeruginosa. The presented structures exhibit an unprecedented metal-binding arrangement consistent with half-of-sites activity: one active site contains a single activating Ni2+ ion while the other contains two inactivating Zn2+ ions. \\nStudents from UWA CHEM3007 Undergraduate Unit helped with some of the data collection.",
		"date_modified": "2015-11-04 07:54:53.470903+00:00",
		"datafile_count": 2171,
		"title": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"dataset_count": 11,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"id": 1038
	},
	{
		"datasets": [
			{
				"id": 70918,
				"description": "Australian Synchrotron Proposal ID: 286"
			}
		],
		"description": "We aim to detemine the crystal structures of several micro-crystalline materials and compare the data quality with that obtained recently for the SCrAPS program at the ChemMatCars beamline.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAFC5C6C52>10.4225/52/557FAFC5C6C52</a></strong>",
		"date_modified": "2010-08-26 00:00:00+00:00",
		"datafile_count": 0,
		"title": "Specialist Crystallography at the Australian Synchrotron",
		"dataset_count": 1,
		"authors": [
			"Jensen P",
			"Turner P"
		],
		"id": 865
	},
	{
		"datasets": [
			{
				"id": 70949,
				"description": "Ancillary Files"
			},
			{
				"id": 70948,
				"description": "Images"
			}
		],
		"description": "The dataset contains 127 images collected at BM14 (ESRF Grenoble, France). Resolution of the final dataset is 2.71 A\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9A8469CBA>10.4225/52/557F9A8469CBAF</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 129,
		"title": "Bacterioferritin from Mycobacterium smegmatis contains zinc in its di-nuclear site.",
		"dataset_count": 2,
		"authors": [
			"Janowski R",
			"Auerbach-Nevo T",
			"Weiss MS"
		],
		"id": 876
	},
	{
		"datasets": [
			{
				"id": 70952,
				"description": "Log Files"
			},
			{
				"id": 70951,
				"description": "Low Resolution"
			},
			{
				"id": 70950,
				"description": "High Resolution"
			}
		],
		"description": "In mammals succinic semialdehyde dehydrogenase (SSADH) plays an essential role in the metabolism of the inhibitory neurotransmitter \\u2248\\u00ed\\u201a\\u00e2\\u2022-aminobutyric acid (GABA) to succinic acid (SA).  Deficiency of SSADH in humans results in elevated levels of GABA and \\u2248\\u00ed\\u201a\\u00e2\\u2022-Hydroxybutyric acid (GHB) which leads to psychomotor retardation, muscular hypotonia, non-progressive ataxia and seizures.  In Escherichia coli, two genetically distinct forms of SSADHs had been described which are essential for preventing accumulation of toxic levels of succinic semialdehyde (SSA) in cells.  Here we structurally characterise SSADH encoded by the E coli gabD gene and compare these data with the structure of human SSADH.  Interestingly, in contrast to the human enzyme in the E. coli SSADH structure, electron density for the complete NADP+ cofactor in the binding sites is clearly evident; these data in particular revealing how the nicotinamide ring of the co-facto is positioned in each active site. Furthermore, our structural data suggest that a deletion of three amino acids in E. coli SSADH permits this enzyme to use NADP+ whereas in contrast the human enzyme utilises NAD+.  Finally, the structure of E. coli SSADH gives additional insight into human mutations that result in disease.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAF18B53C8>10.4225/52/557FAF18B53C8</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 1529,
		"title": "The X-ray crystal structure of Escherichia coli succinic semialdehyde dehydrogenase; structural insights into NADP+ / enzyme interactions",
		"dataset_count": 3,
		"authors": [
			"Langendorf CG",
			"Key TLG",
			"Fenalti G",
			"Kan WT",
			"Buckle AM",
			"Caradoc-Davies T",
			"Tuck KL",
			"Law RHP",
			"Whisstock JC"
		],
		"id": 877
	},
	{
		"datasets": [
			{
				"id": 70970,
				"description": "xtal8 small oscillation angle"
			},
			{
				"id": 70966,
				"description": "xtal8 small oscillation angle"
			}
		],
		"description": "Current therapeutics and prophylactics for malaria are under severe challenge as a result of the rapid emergence of drug-resistant parasites. The human malaria parasite Plasmodium falciparum expresses two neutral aminopeptidases, PfA-M1 and PfA-M17, which function in regulating the intracellular pool of amino acids required for growth and development inside the red blood cell. These enzymes are essential for parasite viability and are validated therapeutic targets. We previously reported the X-ray crystal structure of the monomeric PfA-M1 and proposed a mechanism for substrate entry and free amino acid release from the active site. Here we present the X-ray crystal structure of the hexameric leucine aminopeptidase, PfA-M17, alone and in complex with two inhibitors with anti-malarial activity. The six active sites of the PfA-M17 hexamer are arranged in a disc-like fashion so that they are orientated inwards to form a central catalytic cavity; flexible loops that sit at each of the six entrances to the catalytic cavern function to regulate substrate access. In stark contrast to PfA-M1, PfA-M17 has a narrow and hydrophobic primary specificity pocket which accounts for its highly restricted substrate specificity. We also explicate the essential roles for the metal-binding centres in these enzymes (two in PfA-M17 and one in PfA-M1) in both substrate and drug binding. Our detailed understanding of the PfA-M1 and PfA-M17 active sites now permits a rational approach in the development of a novel class of two-target and/or combination anti-malarial therapy. \\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAC0928E3D>10.4225/52/557FAC0928E3D</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 4157,
		"title": "Structure of the Plasmodium falciparum PfA-M17",
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan"
		],
		"id": 881
	},
	{
		"datasets": [
			{
				"id": 70996,
				"description": "Ancillary Files"
			},
			{
				"id": 70995,
				"description": "Native"
			}
		],
		"description": "Most serpins are associated with protease inhibition, and their ability to form loop-sheet polymers is linked to conformational disease and the human serpinopathies. Here we describe the structural and functional dissection of how a unique serpin, the non-histone architectural protein, MENT (Myeloid and Erythroid Nuclear Termination stage-specific protein), participates in DNA and chromatin condensation. Our data suggest that MENT contains at least two distinct DNA-binding sites, consistent with its simultaneous binding to the two closely juxtaposed linker DNA segments on a nucleosome. Remarkably, our studies suggest that the reactive centre loop, a region of the MENT molecule essential for chromatin bridging in vivo and in vitro, is able to mediate formation of a loop-sheet oligomer. These data provide mechanistic insight into chromatin compaction by a non-histone architectural protein and suggest how the structural plasticity of serpins has adapted to mediate physiological, rather than pathogenic, loop-sheet linkages.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F99816F76F>10.4225/52/557F99816F76F</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 299,
		"title": "X-ray crystal structure of MENT: evidence for functional loop-sheet polymers in chromatin condensation.",
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Irving JA",
			"McGowan S",
			"Whisstock JC"
		],
		"id": 889
	},
	{
		"datasets": [
			{
				"id": 71008,
				"description": "M1_3EBG Structure of the M1 Alanylaminopeptidase from malaria<br/>9_refmac5_final.log Final refinement log file. Refmac v5.2.0019<br/>80_scala.log Processing log file from SCALA<br/>p212121_scala1.mtz Processed data<br/><br/>M1_BES_3EBH Structure of the M1 Alanylaminopeptidase from malaria complexed with bestatin <br/>4_refmac5_finalBES.log Final refinement log file. Refmac v5.2.0019<br/>41_scala.log Processing log file from SCALA<br/>xtal8c_p212121_scala1.mtz Processed data<br/><br/>M1_Co4_3EBI Structure of the M1 Alanylaminopeptidase from malaria complexed with the phosphinate dipeptide analog<br/>.osc files Raw image files. In-house Rotating Anode (Rigaku RAxis IV)<br/>12_refmac5_final Co4.log Final refinement log file. Refmac v5.2.0019<br/>45_scala.log Processing log file from SCALA<br/>3EBIco4_p212121_scala1.mtz Processed data"
			},
			{
				"id": 71007,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria complexed with bestatin"
			},
			{
				"id": 71006,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria complexed with the phosphinate dipeptide analog"
			},
			{
				"id": 71003,
				"description": "Structure of the M1 Alanylaminopeptidase from malaria"
			}
		],
		"description": "This experiment holds 3EBG, 3EBH and 3EBI.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9BF05DBAC>10.4225/52/557F9BF05DBAC</a></strong>\\r\\n\\r\\n",
		"date_modified": "2010-04-30 00:00:00+00:00",
		"datafile_count": 1117,
		"title": "Structural basis for the inhibition of the essential Plasmodium falciparum M1 neutral aminopeptidase",
		"dataset_count": 4,
		"authors": [
			"Buckle AM",
			"Porter CJ",
			"Whisstock JC",
			"McGowan S"
		],
		"id": 893
	},
	{
		"datasets": [
			{
				"id": 71017,
				"description": "AprV2.C141S"
			},
			{
				"id": 71016,
				"description": "AprB2"
			},
			{
				"id": 71015,
				"description": "AprV2"
			},
			{
				"id": 71013,
				"description": "AprV2.C141S"
			},
			{
				"id": 71010,
				"description": "AprB2"
			},
			{
				"id": 71009,
				"description": "AprV2"
			}
		],
		"description": "Many bacterial pathogens produce extracellular proteases that are involved in the degradation of the host extracellular matrix. Dichelobacter nodosus, which causes ovine footrot, is one such pathogen, Mutagenesis and virulence studies revealed that AprV2, one of three secreted subtilisin-like D.\\u00ac\\u2020nodosus proteases, is required for virulence. Our work challenges the previous hypothesis that the elastase activity of AprV2 is important for disease progression, since aprV2 mutants were virulent when complemented with a variant with impaired elastase activity. These data reveal that an unusual extended disulphide-tethered loop functions as an exosite that governs the ability of AprV2 to degrade insoluble extracellular matrix components. The disulphide bond and Tyr92, located at the exposed end of the loop, were functionally important. Bioinformatics suggests that other pathogens utilize a similar mechanism, providing a new paradigm for understanding the role of proteases in disease.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FB124F07CC>10.4225/52/557FB124F07CC</a></strong>",
		"date_modified": "2010-11-25 00:00:00+00:00",
		"datafile_count": 1624,
		"title": "Virulence-associated subtilisin-like proteases that use a novel disulphide-tethered exosite to mediate substrate specificity",
		"dataset_count": 6,
		"authors": [
			"Kennan RM",
			"Wong W",
			"Dhungyel O",
			"Han X",
			"Wong D",
			"Parker D",
			"Rosado CJ",
			"Law RHP",
			"McGowan S",
			"Reeve SB",
			"Levina V",
			"Powers GA",
			"Pike RN",
			"Bottomley SP",
			"Smith AI",
			"Marsh I",
			"Whittington RJ",
			"Whisstock JC",
			"Porter CJ",
			"Rood JI"
		],
		"id": 894
	},
	{
		"datasets": [
			{
				"id": 71036,
				"description": "3T8W_Dataset2"
			},
			{
				"id": 71035,
				"description": "3T8W_dataset1"
			}
		],
		"description": "Malaria causes worldwide morbidity and mortality, and while chemotherapy remains an excellent means of malaria control, drug-resistant parasites necessitate the discovery of new antimalarials. Peptidases are a promising class of drug targets and perform several important roles during the P. falciparum erythrocytic life cycle. Herein, we report a multidisciplinary effort combining activity-based protein profiling, biochemical, and peptidomic approaches to functionally analyze two genetically essential P. falciparum metallo-aminopeptidases (MAPs), PfA-M1 and Pf-LAP. Through the synthesis of a suite of activity-based probes (ABPs) based on the general MAP inhibitor scaffold, bestatin, we generated specific ABPs for these two enzymes. Specific inhibition of PfA-M1 caused swelling of the parasite digestive vacuole and prevented proteolysis of hemoglobin (Hb)-derived oligopeptides, likely starving the parasite resulting in death. In contrast, inhibition of Pf-LAP was lethal to parasites early in the lifecycle, prior to the onset of Hb degradation suggesting that Pf-LAP has an essential role outside of Hb digestion.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9D50D4B61>10.4225/52/557F9D50D4B61</a></strong>",
		"date_modified": "2011-09-23 00:00:00+00:00",
		"datafile_count": 1752,
		"title": "A bestatin-based chemical biology strategy reveals distinct roles for malaria M1- and M17-family aminopeptidases",
		"dataset_count": 2,
		"authors": [
			"McGowan S",
			"Klemba M",
			"Greenbaum DC"
		],
		"id": 897
	},
	{
		"datasets": [
			{
				"id": 70997,
				"description": "Images"
			}
		],
		"description": "The serpin alpha(2)-antiplasmin (SERPINF2) is the principal inhibitor of plasmin and inhibits fibrinolysis. Accordingly, alpha(2)-antiplasmin deficiency in humans results in uncontrolled fibrinolysis and a bleeding disorder. alpha(2)-antiplasmin is an unusual serpin, in that it contains extensive N- and C-terminal sequences flanking the serpin domain. The N-terminal sequence is crosslinked to fibrin by factor XIIIa, whereas the C-terminal region mediates the initial interaction with plasmin. To understand how this may happen, we have determined the 2.65A X-ray crystal structure of an N-terminal truncated murine alpha(2)-antiplasmin. The structure reveals that part of the C-terminal sequence is tightly associated with the body of the serpin. This would be anticipated to position the flexible plasmin-binding portion of the C-terminus in close proximity to the serpin Reactive Center Loop where it may act as a template to accelerate serpin/protease interactions.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F844208778>10.4225/52/557F844208778</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 241,
		"title": "X-ray crystal structure of the fibrinolysis inhibitor {alpha}2-antiplasmin",
		"dataset_count": 1,
		"authors": [
			"Law RHP",
			"Whisstock JC"
		],
		"id": 890
	},
	{
		"datasets": [
			{
				"id": 70988,
				"description": "Ancillary Files"
			},
			{
				"id": 70987,
				"description": "Inhouse"
			}
		],
		"description": "Lysins are peptidoglycan hydrolases that are produced by bacteriophage and act to lyse the bacterial host cell wall during progeny phage release. Here, we describe the structure and function of a novel bacteriophage-derived lysin, PlyB, which displays potent lytic activity against the Bacillus anthracis-like strain ATCC 4342. This molecule comprises an N-terminal catalytic domain (PlyB(cat)) and a C-terminal bacterial SH3-like domain, SH3b. It is shown that both domains are required for effective catalytic activity against ATCC 4342. Further, PlyB has specific activity comparable to the phage lysin PlyG, an amidase being developed as a therapeutic against anthrax. In contrast to PlyG, however, the 1.6 A X-ray crystal structure of PlyB(cat) reveals that the catalytic domain adopts the glycosyl hydrolase (GH)-25, rather than phage T7 lysozyme-like fold. PlyB therefore represents a new class of anthrax lysin and a new defensive tool in the armament against anthrax-mediated bioterrorism.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA83355A9D>10.4225/52/557FA83355A9D</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 667,
		"title": "The 1.6 A Crystal Structure of the Catalytic Domain of PlyB, a Bacteriophage Lysin Active Against Bacillus anthracis.",
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Porter CJ",
			"Whisstock JC"
		],
		"id": 885
	},
	{
		"datasets": [
			{
				"id": 70960,
				"description": "Ancillary Files"
			},
			{
				"id": 70959,
				"description": "Native 2.0 Angstroms Collection"
			},
			{
				"id": 70958,
				"description": "Derivative (KI) in-house"
			},
			{
				"id": 70957,
				"description": "Derivative (KPtCl) inhouse Collection date"
			},
			{
				"id": 70954,
				"description": "Derivative (EMSA) in-house"
			},
			{
				"id": 70953,
				"description": "Native 2.5 Angstroms Collection"
			}
		],
		"description": "Protein crystallography raw diffraction images and unmerged reflection intensities Collection size: 36.1 GB Number of datasets: 5 Citation: Rosado et. al. (2007) A common fold mediates vertebrate defense and bacterial attack. Science. In Press.\\r\\n\\r\\nProteins containing membrane attack complex/perforin (MACPF) domains play important roles in vertebrate immunity, embryonic development, and neural-cell migration. In vertebrates, the ninth component of complement and perforin form oligomeric pores that lyse bacteria and kill virus-infected cells, respectively. However, the mechanism of MACPF function is unknown. We determined the crystal structure of a bacterial MACPF protein, Plu-MACPF from Photorhabdus luminescens, to 2.0 angstrom resolution. The MACPF domain reveals structural similarity with poreforming cholesterol-dependent cytolysins (CDCs) from Gram-positive bacteria. This suggests that lytic MACPF proteins may use a CDC-like mechanism to form pores and disrupt cell membranes. Sequence similarity between bacterial and vertebrate MACPF domains suggests that the fold of the CDCs, a family of proteins important for bacterial pathogenesis, is probably used by vertebrates for defense against infection.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9B41BBAC7>10.4225/52/557F9B41BBAC7</a></strong>\\r\\n",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2276,
		"title": "A common fold mediates vertebrate defense and bacterial attack",
		"dataset_count": 6,
		"authors": [
			"Buckle AM",
			"Dunstone MA",
			"Law RHP",
			"Whisstock JC"
		],
		"id": 878
	},
	{
		"datasets": [
			{
				"id": 54332,
				"description": "Raw data for D3_2"
			},
			{
				"id": 54331,
				"description": "Raw data for D3_1"
			}
		],
		"description": "The Zn inactive class of glyoxalase I (Glo1) enzymes are metalloenzymes that are typically homodimeric with two metal-dependent active sites. While the two active sites share identical amino acid composition, this class of enzyme is optimally active with only one metal per homodimer. We have determined the X-ray crystal structure of GloA2, one of the Zn inactive Glo1 enzymes from Pseudomonas aeruginosa. The presented structures exhibit an unprecedented metal-binding arrangement consistent with half-of-sites activity: one active site contains a single activating Ni2+ ion while the other contains two inactivating Zn2+ ions. \\nStudents from UWA CHEM3007 Undergraduate Unit helped with some of the data collection.",
		"date_modified": "2015-11-05 22:14:06.262874+00:00",
		"datafile_count": 720,
		"title": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"dataset_count": 2,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"id": 1037
	},
	{
		"datasets": [
			{
				"id": 70990,
				"description": "Ancillary Files"
			},
			{
				"id": 70989,
				"description": "Native"
			}
		],
		"description": "Haloacid dehalogenases catalyse the removal of halides from organic haloacids and are of interest for bioremediation and for their potential use in the synthesis of industrial chemicals. We present the crystal structure of the homodimer DehI from Pseudomonas putida strain PP3, the first structure of a group I alpha-haloacid dehalogenase that can process both l- and d-substrates. The structure shows that the DehI monomer consists of two domains of approximately 130 amino acids that have approximately 16% sequence identity yet adopt virtually identical and unique folds that form a pseudo-dimer. Analysis of the active site reveals the likely binding mode of both l- and d-substrates with respect to key catalytic residues. Asp189 is predicted to activate a water molecule for nucleophilic attack of the substrate chiral centre resulting in an inversion of configuration of either l- or d-substrates in contrast to d-only enzymes. These details will assist with future bioengineering of dehalogenases.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9DE24FE22>10.4225/52/557F9DE24FE22</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 516,
		"title": "The crystal structure of DehI reveals a new alpha-haloacid dehalogenase fold and active-site mechanism.",
		"dataset_count": 2,
		"authors": [
			"Wilce JW",
			"Schmidberger MCJ"
		],
		"id": 886
	},
	{
		"datasets": [
			{
				"id": 71034,
				"description": "Ancillary Files"
			},
			{
				"id": 71032,
				"description": "APS Source 80 Deg 5 Sec 0.5 Deg - p212121"
			},
			{
				"id": 71030,
				"description": "APS Source 125 Deg (includes xfiles, denzo)"
			}
		],
		"description": "Serpins fold into a native metastable state and utilize a complex conformational change to inhibit target proteases. An undesirable result of this conformational flexibility is that most inhibitory serpins are heat sensitive, forming inactive polymers at elevated temperatures. However, the prokaryote serpin, thermopin, from Thermobifida fusca is able to function in a heated environment. We have determined the 1.8 A x-ray crystal structure of thermopin in the native, inhibitory conformation. A structural comparison with the previously determined 1.5 A structure of cleaved thermopin provides detailed insight into the complex mechanism of conformational change in serpins. Flexibility in the shutter region and electrostatic interactions at the top of the A beta-sheet (the breach) involving the C-terminal tail, a unique structural feature of thermopin, are postulated to be important for controlling inhibitory activity and triggering conformational change, respectively, in the native state. Here we have discussed the structural basis of how this serpin reconciles the thermodynamic instability necessary for function with the stability required to withstand elevated temperatures.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA0CF23E0E>10.4225/52/557FA0CF23E0E</a></strong>",
		"date_modified": "2011-09-12 00:00:00+00:00",
		"datafile_count": 456,
		"title": "The high resolution crystal structure of a native thermostable serpin reveals the complex mechanism underpinning the stressed to relaxed transition.",
		"dataset_count": 3,
		"authors": [
			"Bottomley SP",
			"Buckle AM",
			"Butcher RE",
			"Cabrita LD",
			"Fulton KF",
			"Irving JA",
			"Lesk AM",
			"Reeve SL",
			"Rossjohn J",
			"Smith I",
			"Whisstock JC"
		],
		"id": 896
	},
	{
		"datasets": [
			{
				"id": 70994,
				"description": "Native"
			}
		],
		"description": "Proteases act in important homeostatic pathways and are tightly regulated. Here, we report an unusual structural mechanism of regulation observed by the 2.5-A X-ray crystal structure of the serine protease, granzyme C. Although the active-site triad residues adopt canonical conformations, the oxyanion hole is improperly formed, and access to the primary specificity (S1) pocket is blocked through a reversible rearrangement involving Phe-191. Specifically, a register shift in the 190-strand preceding the active-site serine leads to Phe-191 filling the S1 pocket. Mutation of a unique Glu-Glu motif at positions 192-193 unlocks the enzyme, which displays chymase activity, and proteomic analysis confirms that activity of the wild-type protease can be released through interactions with an appropriate substrate. The 2.5-A structure of the unlocked enzyme reveals unprecedented flexibility in the 190-strand preceding the active-site serine that results in Phe-191 vacating the S1 pocket. Overall, these observations describe a broadly applicable mechanism of protease regulation that cannot be predicted by template-based modeling or bioinformatic approaches alone.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA4CCF0DE8>10.4225/52/557FA4CCF0DE8</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 132,
		"title": "The structure of granzyme C reveals an unusual mechanism of protease auto-inhibition",
		"dataset_count": 1,
		"authors": [
			"Buckle AM",
			"Kaiserman D",
			"Law RHP",
			"Whisstock JC",
			"Bird PI"
		],
		"id": 888
	},
	{
		"datasets": [
			{
				"id": 70962,
				"description": "Ancillary Files"
			},
			{
				"id": 70961,
				"description": "Native 2.0 Angstroms"
			}
		],
		"description": "The scabies mite (Sarcoptes scabiei) is a parasitic mite responsible for major morbidity in disadvantaged communities and immuno-compromised patients worldwide.   In addition to the physical discomfort caused by the disease, scabies infestations facilitate infection by Streptococcal species via skin lesions, resulting in a high prevalence of rheumatic fever/heart disease in affected communities. The scabies mite produces 33 proteins that are closely related to the dust mite group 3 allergen and belong to the S1-like protease family (chymotrypsin-like).  However, all but one of these molecules contain mutations in the conserved active-site catalytic triad that are predicted to render them catalytically inactive. These molecules are thus termed Scabies Mite Inactivated Protease Paralogues (SMIPPs). The precise function of SMIPPs remains unclear. However, it has been suggested that these proteins may function by binding and protecting target substrates from cleavage by host immune proteases, thus preventing the host from mounting an effective immune challenge. In order to begin to understand the structural basis for SMIPP function, we solved the crystal structures of SMIPP-S-I1 and SMIPP-S-D1 at 1.85 and 2.0 A resolution respectively. Both structures adopt the characteristic serine protease fold, albeit with large structural variations over much of the molecule.  In both structures, mutations in the catalytic triad together with occlusion of the S1 subsite by a conserved Tyr200 residue is predicted to block substrate ingress.  Accordingly, we show that both proteases lack catalytic function.  Attempts to restore function (via site directed mutagenesis of catalytic residues as well as Tyr200) were unsuccessful. Taken together, these data suggest that SMIPPs have lost the ability to bind substrates in a classical canonical fashion, and instead have evolved alternative functions in the lifecycle of the Scabies mite.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA7A6C8287>10.4225/52/557FA7A6C8287</a></strong>\\r\\n",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 126,
		"title": "Crystal structure of scabies mite inactivated protease paralogue S-D1 (SMIPP-S-D1)",
		"dataset_count": 2,
		"authors": [
			"Buckle AM"
		],
		"id": 879
	},
	{
		"datasets": [
			{
				"id": 54222,
				"description": "Raw data for B5_2"
			},
			{
				"id": 54221,
				"description": "Raw data for B5_1"
			},
			{
				"id": 54203,
				"description": "Raw data for F6_1"
			}
		],
		"description": "The Zn inactive class of glyoxalase I (Glo1) enzymes are metalloenzymes that are typically homodimeric with two metal-dependent active sites. While the two active sites share identical amino acid composition, this class of enzyme is optimally active with only one metal per homodimer. We have determined the X-ray crystal structure of GloA2, one of the Zn inactive Glo1 enzymes from Pseudomonas aeruginosa. The presented structures exhibit an unprecedented metal-binding arrangement consistent with half-of-sites activity: one active site contains a single activating Ni2+ ion while the other contains two inactivating Zn2+ ions. \\nStudents from UWA CHEM3007 Undergraduate Unit helped with some of the data collection.",
		"date_modified": "2015-10-28 03:36:10.728308+00:00",
		"datafile_count": 900,
		"title": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"dataset_count": 3,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"id": 1035
	},
	{
		"datasets": [
			{
				"id": 54325,
				"description": "Raw data for C7_Znlow_1"
			},
			{
				"id": 54324,
				"description": "Raw data for C7_Znhigh_1"
			},
			{
				"id": 54229,
				"description": "Raw data for B7_3"
			},
			{
				"id": 54228,
				"description": "Raw data for B7_2"
			},
			{
				"id": 54227,
				"description": "Raw data for B7_1"
			}
		],
		"description": "The Zn inactive class of glyoxalase I (Glo1) enzymes are metalloenzymes that are typically homodimeric with two metal-dependent active sites. While the two active sites share identical amino acid composition, this class of enzyme is optimally active with only one metal per homodimer. We have determined the X-ray crystal structure of GloA2, one of the Zn inactive Glo1 enzymes from Pseudomonas aeruginosa. The presented structures exhibit an unprecedented metal-binding arrangement consistent with half-of-sites activity: one active site contains a single activating Ni2+ ion while the other contains two inactivating Zn2+ ions. \\nStudents from UWA CHEM3007 Undergraduate Unit helped with some of the data collection.",
		"date_modified": "2015-11-05 22:13:07.722484+00:00",
		"datafile_count": 1800,
		"title": "The crystal structure of a homodimeric Pseudomonas glyoxalase I enzyme reveals asymmetric metallation commensurate with half-of-sites activity",
		"dataset_count": 5,
		"authors": [
			"Rohan Bythell-Douglas",
			"Charles Bond"
		],
		"id": 1036
	},
	{
		"datasets": [
			{
				"id": 70984,
				"description": "zinc soak unbound"
			},
			{
				"id": 70983,
				"description": "zinc soak unbound"
			}
		],
		"description": "Current therapeutics and prophylactics for malaria are under severe challenge as a result of the rapid emergence of drug-resistant parasites. The human malaria parasite Plasmodium falciparum expresses two neutral aminopeptidases, PfA-M1 and PfA-M17, which function in regulating the intracellular pool of amino acids required for growth and development inside the red blood cell. These enzymes are essential for parasite viability and are validated therapeutic targets. We previously reported the X-ray crystal structure of the monomeric PfA-M1 and proposed a mechanism for substrate entry and free amino acid release from the active site. Here we present the X-ray crystal structure of the hexameric leucine aminopeptidase, PfA-M17, alone and in complex with two inhibitors with anti-malarial activity. The six active sites of the PfA-M17 hexamer are arranged in a disc-like fashion so that they are orientated inwards to form a central catalytic cavity; flexible loops that sit at each of the six entrances to the catalytic cavern function to regulate substrate access. In stark contrast to PfA-M1, PfA-M17 has a narrow and hydrophobic primary specificity pocket which accounts for its highly restricted substrate specificity. We also explicate the essential roles for the metal-binding centres in these enzymes (two in PfA-M17 and one in PfA-M1) in both substrate and drug binding. Our detailed understanding of the PfA-M1 and PfA-M17 active sites now permits a rational approach in the development of a novel class of two-target and/or combination anti-malarial therapy. \\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FACDB8FE6C>10.4225/52/557FACDB8FE6C</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 1800,
		"title": "Structure of the Plasmodium falciparum PfA-M17_ZnZn",
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan"
		],
		"id": 884
	},
	{
		"datasets": [
			{
				"id": 70992,
				"description": "Ancillary Files"
			},
			{
				"id": 70991,
				"description": "Inhouse"
			}
		],
		"description": "The mechanism of binding of thyroid hormones by the transport protein transthyretin (TTR) in vertebrates is structurally well characterised. However, a homologous family of transthyretin-like proteins (TLPs) present in bacteria as well as eukaryotes do not bind thyroid hormones, instead they are postulated to perform a role in the purine degradation pathway and function as 5-hydroxyisourate hydrolases. Here we describe the 2.5 Angstroms X-ray crystal structure of the TLP from the Gram-negative bacterium Salmonella dublin, and compare and contrast its structure with vertebrate TTRs. The overall architecture of the homotetramer is conserved and, despite low sequence homology with vertebrate TTRs, structural differences within the monomer are restricted to flexible loop regions. However, sequence variation at the dimer-dimer interface has profound consequences for the ligand binding site and provides a structural rationalisation for the absence of thyroid hormone binding affinity in bacterial TLPs: the deep, negatively charged thyroxine-binding pocket that characterises vertebrate TTR contrasts with a shallow and elongated, positively charged cleft in S. dublin TLP. We have demonstrated that Sdu_TLP is a 5-hydroxyisourate hydrolase. Furthermore, using site-directed mutagenesis, we have identified three conserved residues located in this cleft that are critical to the enzyme activity. Together our data reveal that the active site of Sdu_TLP corresponds to the thyroxine binding site in TTRs.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9F15E68E5>10.4225/52/557F9F15E68E5</a></strong>\\r\\n",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 186,
		"title": "The Crystal Structure of the Transthyretin-like Protein from Salmonella dublin, a Prokaryote 5-Hydroxyisourate Hydrolase",
		"dataset_count": 2,
		"authors": [
			"Buckle AM",
			"Law RHP",
			"Whisstock JC"
		],
		"id": 887
	},
	{
		"datasets": [
			{
				"id": 70980,
				"description": "PfA-M17 with Compound4 bound in active sites"
			},
			{
				"id": 70978,
				"description": "PfA-M17 with Compound4 bound in active sites"
			},
			{
				"id": 70977,
				"description": "PfA-M17 with Compound4 bound in active sites"
			},
			{
				"id": 70976,
				"description": "PfA-M17 with Compound 4 bound in active site"
			}
		],
		"description": "Current therapeutics and prophylactics for malaria are under severe challenge as a result of the rapid emergence of drug-resistant parasites. The human malaria parasite Plasmodium falciparum expresses two neutral aminopeptidases, PfA-M1 and PfA-M17, which function in regulating the intracellular pool of amino acids required for growth and development inside the red blood cell. These enzymes are essential for parasite viability and are validated therapeutic targets. We previously reported the X-ray crystal structure of the monomeric PfA-M1 and proposed a mechanism for substrate entry and free amino acid release from the active site. Here we present the X-ray crystal structure of the hexameric leucine aminopeptidase, PfA-M17, alone and in complex with two inhibitors with anti-malarial activity. The six active sites of the PfA-M17 hexamer are arranged in a disc-like fashion so that they are orientated inwards to form a central catalytic cavity; flexible loops that sit at each of the six entrances to the catalytic cavern function to regulate substrate access. In stark contrast to PfA-M1, PfA-M17 has a narrow and hydrophobic primary specificity pocket which accounts for its highly restricted substrate specificity. We also explicate the essential roles for the metal-binding centres in these enzymes (two in PfA-M17 and one in PfA-M1) in both substrate and drug binding. Our detailed understanding of the PfA-M1 and PfA-M17 active sites now permits a rational approach in the development of a novel class of two-target and/or combination anti-malarial therapy. \\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAC718F196>10.4225/52/557FAC718F196</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2742,
		"title": "Structure of the Plasmodium falciparum PfA-M17_Co4",
		"dataset_count": 4,
		"authors": [
			"Sheena McGowan"
		],
		"id": 883
	},
	{
		"datasets": [
			{
				"id": 71029,
				"description": "mouse_perforin_R213E_I"
			},
			{
				"id": 71026,
				"description": "mouse_perforin_R213E_Ir"
			},
			{
				"id": 71024,
				"description": "mouse_perforin_R213E_Hg"
			},
			{
				"id": 71022,
				"description": "mouse_perforin_R213E_Native2"
			},
			{
				"id": 71019,
				"description": "mouse_perforin_R213E_Native1"
			},
			{
				"id": 71018,
				"description": "Scala Logs & BUSTER Output"
			}
		],
		"description": "Natural killer cells and cytotoxic T lymphocytes accomplish the critically important function of killing virus-infected and neoplastic cells. They do this by releasing the pore-forming protein perforin and granzyme proteases from cytoplasmic granules into the cleft formed between the abutting killer and target cell membranes. Perforin, a 67-kilodalton multidomain protein, oligomerizes to form pores that deliver the pro-apoptopic granzymes into the cytosol of the target cell. The importance of perforin is highlighted by the fatal consequences of congenital perforin deficiency, with more than 50 different perforin mutations linked to familial haemophagocytic lymphohistiocytosis (type 2 FHL). Here we elucidate the mechanism of perforin pore formation by determining the X-ray crystal structure of monomeric murine perforin, together with a cryo-electron microscopy reconstruction of the entire perforin pore. Perforin is a thin \\u2018key-shaped\\u2019 molecule, comprising an amino-terminal membrane attack complex perforin-like (MACPF)/cholesterol dependent cytolysin (CDC) domain followed by an epidermal growth factor (EGF) domain that, together with the extreme carboxy-terminal sequence, forms a central shelf-like structure. A C-terminal C2 domain mediates initial, Ca2+-dependent membrane binding. Most unexpectedly, however, cryo-electron microscopy reveals that the orientation of the perforin MACPF domain in the pore is inside-out relative to the subunit arrangement in CDCs. These data reveal remarkable flexibility in the mechanism of action of the conserved MACPF/CDC fold and provide new insights into how related immune defence molecules such as complement proteins assemble into pores.</abstract>\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA38E11655>10.4225/52/557FA38E11655</a></strong>",
		"date_modified": "2011-05-27 00:00:00+00:00",
		"datafile_count": 5139,
		"title": "The structural basis for membrane binding and pore formation by lymphocyte perforin",
		"dataset_count": 6,
		"authors": [
			"Ruby Law",
			"Natalya Lukoyanova",
			"Ilia Voskoboinik",
			"Tom Caradoc-Davies",
			"Katherine Baran",
			"Michelle A. Dunstone",
			"Michael E. D\\u2019Angelo",
			"Fasseli Coulibaly",
			"Sandra Verschoor",
			"Kylie A. Browne",
			"Annette Ciccone",
			"Michael J. Kuiper",
			"Phillip I. Bird",
			"Joseph A. Trapani",
			"Helen R. Saibil",
			"James C. Whisstock"
		],
		"id": 895
	},
	{
		"datasets": [
			{
				"id": 70971,
				"description": "PfA-M17 with Bestatin bound in active site"
			}
		],
		"description": "Current therapeutics and prophylactics for malaria are under severe challenge as a result of the rapid emergence of drug-resistant parasites. The human malaria parasite Plasmodium falciparum expresses two neutral aminopeptidases, PfA-M1 and PfA-M17, which function in regulating the intracellular pool of amino acids required for growth and development inside the red blood cell. These enzymes are essential for parasite viability and are validated therapeutic targets. We previously reported the X-ray crystal structure of the monomeric PfA-M1 and proposed a mechanism for substrate entry and free amino acid release from the active site. Here we present the X-ray crystal structure of the hexameric leucine aminopeptidase, PfA-M17, alone and in complex with two inhibitors with anti-malarial activity. The six active sites of the PfA-M17 hexamer are arranged in a disc-like fashion so that they are orientated inwards to form a central catalytic cavity; flexible loops that sit at each of the six entrances to the catalytic cavern function to regulate substrate access. In stark contrast to PfA-M1, PfA-M17 has a narrow and hydrophobic primary specificity pocket which accounts for its highly restricted substrate specificity. We also explicate the essential roles for the metal-binding centres in these enzymes (two in PfA-M17 and one in PfA-M1) in both substrate and drug binding. Our detailed understanding of the PfA-M1 and PfA-M17 active sites now permits a rational approach in the development of a novel class of two-target and/or combination anti-malarial therapy. \\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAB9D22BEA>10.4225/52/557FAB9D22BEA</a></strong>",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 2644,
		"title": "Structure of the Plasmodium falciparum PfA-M17_BES",
		"dataset_count": 1,
		"authors": [
			"Sheena McGowan"
		],
		"id": 882
	},
	{
		"datasets": [
			{
				"id": 70965,
				"description": "Ancillary Files"
			},
			{
				"id": 70963,
				"description": "Native"
			}
		],
		"description": "DehIVa is a haloacid dehalogenase (EC 3.8.1.2) from the soil and water borne bacterium Burkholderia cepacia MBA4, which belongs to the functionally variable haloacid dehalogenase (HAD) superfamily of enzymes. The haloacid dehalogenases catalyse the removal of halides from haloacids resulting in a hydroxlated product. These enzymes are of interest for their potential to degrade recalcitrant halogenated environmental pollutants and their use in the synthesis of industrial chemicals. The haloacid dehalogenases utilise a nucleophilic attack on the substrate by an aspartic acid residue to form an enzyme-substrate ester bond and concomitantly cleaving of the carbon-halide bond and release of a hydroxylated product following ester hydrolysis. We present the crystal structures of both the substrate-free DehIVa refined to 1.93 A resolution and DehIVa covalently bound to l-2-monochloropropanoate trapped as a reaction intermediate, refined to 2.7 A resolution. Electron density consistent with a previously unidentified yet anticipated water molecule in the active site poised to donate its hydroxyl group to the product and its proton to the catalytic Asp11 is evident. It has been unclear how substrate enters the active site of this and related enzymes. The results of normal mode analysis (NMA) are presented and suggest a means whereby the predicted global dynamics of the enzyme allow for entry of the substrate into the active site. In the context of these results, the possible role of Arg42 and Asn178 in a `lock down` mechanism affecting active site access is discussed. In silico substrate docking of enantiomeric substrates has been examined in order to evaluate the enzymes enantioselectivity.\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557F9CB23E046>10.4225/52/557F9CB23E046</a></strong>\\r\\n",
		"date_modified": "2010-02-26 00:00:00+00:00",
		"datafile_count": 202,
		"title": "Crystal structures of the substrate free-enzyme, and reaction intermediate of the HAD superfamily member, haloacid dehalogenase DehIVa from Burkholderia cepacia MBA4.",
		"dataset_count": 2,
		"authors": [
			"Schmidberger MCJ",
			"Wilce MCJ"
		],
		"id": 880
	},
	{
		"datasets": [
			{
				"id": 71048,
				"description": "Glycoform 1 Humanplasminogen (4DUU)"
			},
			{
				"id": 71046,
				"description": "Glycoform 2 Humanplasminogen (4DUR)"
			}
		],
		"description": "\\r\\nSample diffraction image (Glycoform 2 Humanplasminogen 4DUR: xtal19_1_001.img):<br/><br/>\\r\\n<img style=width: 500px; src=/display/DatafileImage/load/19138556//>\\r\\n<br/><br/>\\r\\n<strong><a href=http://www.cell.com/cell-reports/fulltext/S2211-1247(12)00069-1>Publication (Cell Reports)</a></strong>\\r\\n<br/><br/>\\r\\nPlasminogen is the proenzyme precursor of the primary fibrinolytic protease plasmin. Circulating plasminogen, which comprises a Pan-apple (PAp) domain, five kringle domains (KR1-5), and a serine protease (SP) domain, adopts a closed, activation-resistant conformation. The kringle domains mediate interactions with fibrin clots and cell-surface receptors. These interactions trigger plasminogen to adopt an open form that can be cleaved and converted to plasmin by tissue-type and urokinase-type plasminogen activators. Here, the structure of closed plasminogen reveals that the PAp and SP domains, together with chloride ions, maintain the closed conformation through interactions with the kringle array. Differences in glycosylation alter the position of KR3, although in all structures the loop cleaved by plasminogen activators is inaccessible. The ligand-binding site of KR1 is exposed and likely governs proenzyme recruitment to targets. Furthermore, analysis of our structure suggests that KR5 peeling away from the PAp domain may initiate plasminogen conformational change.\\r\\n<p>\\r\\nThis entry contains two diffraction datasets:\\r\\n<ul>\\r\\n    <li>G1-10 Glycoform 1 Humanplasminogen (PDB ID: <a href=http://www.rcsb.org/pdb/explore/explore.do?structureId=4DUU>4DUU)<br/>\\r\\n        <img src=http://www.rcsb.org/pdb/images/4duu_bio_r_250.jpg/></a>\\r\\n    </li>\\r\\n    <li>Xtal19 Glycoform2 Humanplasminogen (PDB ID: <a href=http://www.rcsb.org/pdb/explore/explore.do?structureId=4DUR>4DUR)<br/>\\r\\n            <img src=http://www.rcsb.org/pdb/images/4dur_bio_r_250.jpg/></a>\\r\\n    </li>\\r\\n</ul>\\r\\n</p>\\r\\n\\t \\r\\n\\r\\nAutomatically generated on 2015-06-02 06:09:44 by https://github.com/steveandroulakis/mytardis-uploader\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FA955BB3A0>10.4225/52/557FA955BB3A0</a></strong>",
		"date_modified": "2012-08-22 00:00:00+00:00",
		"datafile_count": 572,
		"title": "The X-ray Crystal Structure of Full-length Human Plasminogen",
		"dataset_count": 2,
		"authors": [
			"Ruby H.P. Law",
			"Tom Caradoc-Davies",
			"Nathan Cowieson",
			"Anita J. Horvath",
			"Adam J. Quek",
			"Joanna Amarante Encarnacao",
			"David Steer",
			"Angus Cowan",
			"Qingwei Zhang",
			"Bernadine G.C. Lu",
			"Robert N. Pike",
			"A. Ian Smith",
			"Paul B. Coughlin",
			"James C. Whisstock"
		],
		"id": 899
	},
	{
		"datasets": [
			{
				"id": 71051,
				"description": "M18"
			}
		],
		"description": "\\r\\nThe malarial aminopeptidases have emerged as promising new drug targets for the development of novel anti-malarial drugs.  The M18AAP of Plasmodium falciparum malaria is a metallo-aminopeptidase that we show demonstrates a highly restricted specificity for peptides with an N-terminal Glu or Asp residue. Thus the enzyme may function alongside other aminopeptidases in effecting the complete degradation or turnover of proteins, such as host hemoglobin, which provides a free amino acid pool for the growing parasite. Inhibition of PfM18AAP\\u201a\\u00c4\\u00f4s function using antisense RNA is detrimental to the intra-erythrocytic malaria parasite and, hence, it has been proposed as a potential novel drug target.  We report the X-ray crystal structure of the PfM18AAP aminopeptidase and reveal its complex dodecameric assembly arranged via dimer and trimer units that interact to form a large tetrahedron shape that completely encloses the 12 active sites within a central cavity. The four entry points to the catalytic lumen are each guarded by 12 large flexible loops that could control substrate entry into the catalytic sites. PfM18AAP thus resembles a proteasomal-like machine with multiple active sites able to degrade peptide substrates that enter the central lumen. The Plasmodium enzyme shows significant structural differences around the active site when compared to recently determined structures of its mammalian and human homologs which provides a platform from which a rational approach to inhibitor design of new malaria-specific drugs can begin.\\r\\n\\t \\r\\n\\r\\nAutomatically generated on 2015-06-02 06:32:01 by https://github.com/steveandroulakis/mytardis-uploader\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAAF66B0CF>10.4225/52/557FAAF66B0CF</a></strong>",
		"date_modified": "2012-08-09 00:00:00+00:00",
		"datafile_count": 240,
		"title": "X-ray crystal structure and specificity of the Plasmodium falciparum malaria aminopeptidase PfM18AAP",
		"dataset_count": 1,
		"authors": [
			"Sheena McGowan"
		],
		"id": 900
	},
	{
		"datasets": [
			{
				"id": 71055,
				"description": "PlyC"
			},
			{
				"id": 71052,
				"description": "PlyCB"
			}
		],
		"description": "\\r\\n<strong><a href=http://www.pnas.org/content/109/31/12752>Publication (PNAS)</a></strong>\\r\\n<br/><br/>\\r\\nBacteriophages deploy lysins that degrade the bacterial cell wall and facilitate virus egress from the host. When applied exogenously, these enzymes destroy susceptible microbes and, accordingly, have potential as therapeutic agents. The most potent lysin identified to date is PlyC, an enzyme assembled from two components (PlyCA and PlyCB) that is specific for streptococcal species. Here the structure of the PlyC holoenzyme reveals that a single PlyCA moiety is tethered to a ring-shaped assembly of eight PlyCB molecules. Structure-guided mutagenesis reveals that the bacterial cell wall binding is achieved through a cleft on PlyCB. Unexpectedly, our structural data reveal that PlyCA contains a glycoside hydrolase domain in addition to the previously recognized cysteine, histidine-dependent amidohydrolases/peptidases catalytic domain. The presence of eight cell wall-binding domains together with two catalytic domains may explain the extraordinary potency of the PlyC holoenyzme toward target bacteria.\\r\\n<p>\\r\\nThis entry contains two diffraction datasets:\\r\\n<ul>\\r\\n    <li>PlyCB <a href=http://www.rcsb.org/pdb/explore/explore.do?structureId=4F87>(PDB ID: 4F87)<br/>\\r\\n        <img src=http://www.rcsb.org/pdb/images/4f87_bio_r_250.jpg/></a>\\r\\n    </li>\\r\\n    <li>PlyC <a href=http://www.rcsb.org/pdb/explore/explore.do?structureId=4F88>(PDB ID: 4F88)<br/>\\r\\n            <img src=http://www.rcsb.org/pdb/images/4f88_bio_r_250.jpg/></a>\\r\\n    </li>\\r\\n</ul>\\r\\n</p>\\r\\n\\t \\r\\n\\r\\nAutomatically generated on 2015-06-02 06:39:07 by https://github.com/steveandroulakis/mytardis-uploader\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAA5E63100>10.4225/52/557FAA5E63100</a></strong>",
		"date_modified": "2012-08-22 00:00:00+00:00",
		"datafile_count": 531,
		"title": "X-ray crystal structure of the streptococcal specific phage lysin PlyC",
		"dataset_count": 2,
		"authors": [
			"Sheena McGowan",
			"Ashley Buckle",
			"James Whisstock"
		],
		"id": 901
	},
	{
		"datasets": [
			{
				"id": 71040,
				"description": "Raw data for Compound 7c"
			},
			{
				"id": 71039,
				"description": "Raw data for Compound 7d"
			}
		],
		"description": "The essential malarial PfA-M1 metalloaminopeptidase is a validated drug target that functions in the terminal stages of hemoglobin digestion. The natural product dipeptide mimetic, bestatin, is a potent inhibitor of PfA-M1 and provides an excellent scaffold for the development of novel research tools as well as more effective PfA-M1 inhibitors. Here we present a new, efficient and high yielding protocol for the synthesis of bestatin-derivatives from commercially available natural and unnatural N-Boc-D-amino acids. We developed a diverse library of bestatin derivatives with variants at the sidechain of either the \\u0152\\u00b1-hydroxy-\\u0152\\u2264-amino acid or the adjacent natural \\u0152\\u00b1-amino acid. Surprisingly we found that large aromatic rings at the P1 position resulted in potent inhibition against PfA-M1, while small hydrophobic sidechains were favored at the P1\\u201a\\u00c4\\u00f4 position. These data contrast previous studies that suggested the primary substrate specificity (S1) pocket of the PfA-M1 enzyme is unable to accommodate side-chains much larger than a P1 phenylalanine. To understand these apparently contradictory data, we determined the X-ray crystal structure of the PfA-M1 / bestatin-Tyr(OBzl) complex. The structure revealed a substantial inhibitor-induced rearrangement of the primary loop that forms the S1 pocket that permits accommodation of the bestatin-Tyr(OBzl) inhibitor. These findings are in contrast to most proteases where the S1 pocket is considered to define primary enzyme specificity through substantial rigidity. Taken together, our data provide important insights for the rational design of more potent and selective inhibitors of this enzyme, which may eventually be of therapeutic value for the treatment of malaria.</abstract>\\r\\n\\r\\nTo cite this data use the following DOI: <strong><a href=http://dx.doi.org/10.4225/52/557FAD81B7777>10.4225/52/557FAD81B7777</a></strong>",
		"date_modified": "2012-11-15 00:00:00+00:00",
		"datafile_count": 1440,
		"title": "Synthesis of new (-)-Bestatin-based inhibitor libraries reveals a novel binding mode in the S1 pocket of the essential malaria M1 metalloaminopeptidase.",
		"dataset_count": 2,
		"authors": [
			"Geetha Velmourougane",
			"Michael B. Harbut",
			"Seema Dalal",
			"Sheena McGowan",
			"Christine A. Oellig",
			"James C. Whisstock",
			"Michael Klemba",
			"Doron C. Greenbaum"
		],
		"id": 898
	}
]
function ordinal_suffix_of(i) {
    let j = i % 10,
        k = i % 100;
    if (j === 1 && k !== 11) {
        return i + "st";
    }
    if (j === 2 && k !== 12) {
        return i + "nd";
    }
    if (j === 3 && k !== 13) {
        return i + "rd";
    }
    return i + "th";
}
function htmlToElement(html) {
    let template = document.createElement('template');
    html = html.trim();
    template.innerHTML = html;
    return template.content.firstChild;
}
console.log(experiment_id);
//get data
let exp_data = data.find(o => o.id === experiment_id)
//get description Data
let desc_data = exp_array.find(o => o.id === experiment_id)
console.log(exp_data)
console.log(desc_data)
window.onload = create_elems;
//insert title
function create_elems() {
    //append title
    const title = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > h1 > span")
    title.textContent = exp_data.name
    //append author
    const authors_elem = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > span")
    const authors = exp_data.authors
    authors.forEach(author => {
        //insert elem
        let author_span = document.createElement('span')
        let author_name_span = document.createElement('span')
        author_name_span.setAttribute("style", "white-space: nowrap;")
        if(authors[authors.length - 1] === author){
            author_name_span.textContent = author.trim()
        } else {
            author_name_span.textContent = author + " , "
        }

        author_span.appendChild(author_name_span)
        authors_elem.appendChild(author_span)
    })
	// Description
	const description_holder = document.querySelector("#abstractHolder")
	description_holder.textContent = desc_data.description
    const description_elem = document.querySelector("#abstractText")
	description_elem.textContent = desc_data.description

    //populate description tab
    const inst_elem = document.querySelector("#experiment_description > dl:nth-child(1) > dd")
    inst_elem.textContent = exp_data.description.institution
    const lic_elem = document.querySelector("#experiment_description > dl:nth-child(2) > dd > div > div:nth-child(1) > a")
    lic_elem.textContent = exp_data.description.license.license
    lic_elem.setAttribute("href", exp_data.description.license.url)
	const tar_elem = document.querySelector("#experiment_description > dl.row.download_entire_experiment > dd > a")
	tar_elem.href = `/files/${experiment_id}.tar.gz`
     // admin
    const admin_elem = document.querySelector("#experiment_description > dl:nth-child(3) > dd > a > span")
    admin_elem.textContent = exp_data.description.admins[0]
    // badges
    const ds_count = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > p > span:nth-child(1)")
    ds_count.textContent = exp_data.dataset_count
    const df_count = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > p > span:nth-child(2)")
    df_count.textContent = exp_data.datafile_count
    const exp_size = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > p > span:nth-child(3)")
    exp_size.textContent = filesize(exp_data.size)
    const date = new Date(exp_data.date_modified)
    const dateTimeFormat = new Intl.DateTimeFormat('en', { year: 'numeric', month: 'short', day: '2-digit' })
    let [{ value: month },,{ value: day },,{ value: year }] = dateTimeFormat .formatToParts(date )
    day = ordinal_suffix_of(day)
    const badgeDateSpan = document.querySelector("body > div.container-fluid > div.row > div > div.page-header > p > span:nth-child(4)")
    badgeDateSpan.textContent = `${day} ${month} ${year }`
    //dataset list
    const datasetCount = document.querySelector("#datasets-pane > h3")
    datasetCount.textContent = exp_data.dataset_count + " Datasets"
	//dataset tiles
	const datasetTilesList = document.querySelector("#datasets > ul")
	const dataset_data = exp_data.datasets
	dataset_data.forEach( dataset => {
		//create card elem
		var size = filesize(dataset.size)
		datasetTilesList.append(htmlToElement(`
		<li class="dataset card mb-2 ui-sortable-handle" style="margin-left: 0">
			<div class="dataset-tile thumbnail">
			<div class="card-body">
			<div class="float-left" style="margin-right: 10px">
			<div style="display: flex; text-align: center; font-size: 32px">
			<i class="fa fa-folder-open o-6"></i>
			</div>
			<br>
			</div>
			<div class="float-left" style="margin-right: 10px">
			</div>
			<div class="float-right" style="text-align: right">
			<p>
			<span class="badge badge-info" title="Contains ${dataset.file_count} files">
			<i class="icon-file"></i>&nbsp;
			${dataset.file_count}
			</span>
			</p>
			<p>
			<span class="badge badge-info" title="Dataset size is ">
			${size}
			</span>
			</p>
			</div>
			<p class="">
			<a class="dataset-link">
			${dataset.description}
			</a>
			</p>
			<p style="color:grey;font-size:100%;">Folder:
			 <span style="color:#3a87ad;font-weight:bold;" id="folder">frames</span>
			</p>
			<div style="clear: both"></div>
			</div>
			</div>
		</li>
	`))
	})
	//meta data
	const metaDataPane = document.querySelector("#experiment-tab-metadata")
	const metadataData = exp_data.metadata
	metadataData.forEach(elem => {
		const th_data = elem.name
		const table_div = document.createElement('div')
		table_div.setAttribute("style", "margin-top: 10px")
		const table_elem = document.createElement('table')
		table_elem.setAttribute("class", "parameter_table table table-striped table-bordered protein-data-bank")
		const table_body = document.createElement('tbody')
		table_elem.appendChild(table_body)
		table_body.append(htmlToElement(`
			<tr>
				<th class="schema_name" title="http://synchrotron.org.au/pub/mx/pdb/ (immutable)" colspan="2">
					${th_data}
				</th>
			</tr>		
		`))
		const metadata_values = elem.values
		metadata_values.forEach(value => {
			//apend row
			const row_elem = document.createElement('tr')
			const row_td = document.createElement('td')
			row_td.setAttribute("class", "parameter_name")
			row_td.textContent = value.name
			row_elem.appendChild(row_td)
            //append value
			const value_td = document.createElement('td')
			value_td.setAttribute("class", "parameter_value")
			value_td.textContent = value.value
			row_elem.appendChild(value_td)
			//
			table_body.appendChild(row_elem)
		})


		table_div.appendChild(table_elem)
		metaDataPane.appendChild(table_div)
	})
}
